{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "679b8254-4afa-47e2-98c1-53a3d349c30c",
   "metadata": {},
   "source": [
    "# 240829"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "bbf11f4a-65f7-422c-897f-92c581d73d4c",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "     active environment : None\n",
      "       user config file : /home/dnanexus/.condarc\n",
      " populated config files : \n",
      "          conda version : 23.7.4\n",
      "    conda-build version : not installed\n",
      "         python version : 3.9.16.final.0\n",
      "       virtual packages : __archspec=1=x86_64\n",
      "                          __glibc=2.31=0\n",
      "                          __linux=5.15.0=0\n",
      "                          __unix=0=0\n",
      "       base environment : /opt/conda  (writable)\n",
      "      conda av data dir : /opt/conda/etc/conda\n",
      "  conda av metadata url : None\n",
      "           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64\n",
      "                          https://repo.anaconda.com/pkgs/main/noarch\n",
      "                          https://repo.anaconda.com/pkgs/r/linux-64\n",
      "                          https://repo.anaconda.com/pkgs/r/noarch\n",
      "          package cache : /opt/conda/pkgs\n",
      "                          /home/dnanexus/.conda/pkgs\n",
      "       envs directories : /opt/conda/envs\n",
      "                          /home/dnanexus/.conda/envs\n",
      "               platform : linux-64\n",
      "             user-agent : conda/23.7.4 requests/2.32.0 CPython/3.9.16 Linux/5.15.0-1068-aws ubuntu/20.04.6 glibc/2.31\n",
      "                UID:GID : 0:0\n",
      "             netrc file : None\n",
      "           offline mode : False\n",
      "\n",
      "\n",
      "# packages in environment at /opt/conda:\n",
      "#\n",
      "# Name                    Version                   Build  Channel\n",
      "_libgcc_mutex             0.1                        main  \n",
      "_openmp_mutex             5.1                       1_gnu  \n",
      "aiofiles                  22.1.0             pyhd8ed1ab_0    conda-forge\n",
      "aiosqlite                 0.20.0             pyhd8ed1ab_0    conda-forge\n",
      "annotated-types           0.7.0                    pypi_0    pypi\n",
      "ansiwrap                  0.8.4                    pypi_0    pypi\n",
      "anyio                     3.7.1              pyhd8ed1ab_0    conda-forge\n",
      "argcomplete               1.12.3                   pypi_0    pypi\n",
      "argon2-cffi               23.1.0             pyhd8ed1ab_0    conda-forge\n",
      "argon2-cffi-bindings      21.2.0           py39hb9d737c_2    conda-forge\n",
      "asttokens                 2.4.1              pyhd8ed1ab_0    conda-forge\n",
      "attrs                     24.2.0             pyh71513ae_0    conda-forge\n",
      "babel                     2.14.0             pyhd8ed1ab_0    conda-forge\n",
      "bash-kernel               0.9.3                    pypi_0    pypi\n",
      "beautifulsoup4            4.12.3             pyha770c72_0    conda-forge\n",
      "bleach                    6.1.0              pyhd8ed1ab_0    conda-forge\n",
      "boltons                   23.0.0           py39h06a4308_0  \n",
      "brotlipy                  0.7.0           py39h27cfd23_1003  \n",
      "bzip2                     1.0.8                h5eee18b_6  \n",
      "ca-certificates           2024.7.2             h06a4308_0  \n",
      "certifi                   2024.7.4         py39h06a4308_0  \n",
      "cffi                      1.15.1           py39h5eee18b_3  \n",
      "charset-normalizer        2.0.4              pyhd3eb1b0_0  \n",
      "click                     8.1.7                    pypi_0    pypi\n",
      "coloredlogs               15.0.1                   pypi_0    pypi\n",
      "colormath                 3.0.0                    pypi_0    pypi\n",
      "comm                      0.2.2              pyhd8ed1ab_0    conda-forge\n",
      "conda                     23.7.4           py39hf3d152e_0    conda-forge\n",
      "conda-content-trust       0.1.3            py39h06a4308_0  \n",
      "conda-package-handling    2.0.2            py39h06a4308_0  \n",
      "conda-package-streaming   0.7.0            py39h06a4308_0  \n",
      "contourpy                 1.2.1                    pypi_0    pypi\n",
      "cryptography              39.0.2                   pypi_0    pypi\n",
      "cutadapt                  4.9                      pypi_0    pypi\n",
      "cycler                    0.12.1                   pypi_0    pypi\n",
      "dbus                      1.13.18              hb2f20db_0  \n",
      "debugpy                   1.6.0            py39h5a03fae_0    conda-forge\n",
      "decorator                 4.4.2                    pypi_0    pypi\n",
      "defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge\n",
      "dnaio                     1.2.1                    pypi_0    pypi\n",
      "dxdata                    0.41.0                   pypi_0    pypi\n",
      "dxpy                      0.377.0                  pypi_0    pypi\n",
      "entrypoints               0.4                pyhd8ed1ab_0    conda-forge\n",
      "exceptiongroup            1.2.2              pyhd8ed1ab_0    conda-forge\n",
      "executing                 2.0.1              pyhd8ed1ab_0    conda-forge\n",
      "expat                     2.6.2                h6a678d5_0  \n",
      "fastqc                    0.11.8                        0    bioconda\n",
      "findspark                 2.0.1                    pypi_0    pypi\n",
      "fonttools                 4.53.1                   pypi_0    pypi\n",
      "freetype                  2.12.1               h4a9f257_0  \n",
      "future                    1.0.0                    pypi_0    pypi\n",
      "gdbm                      1.18                 hd4cb3f1_4  \n",
      "glib                      2.78.4               h6a678d5_0  \n",
      "glib-tools                2.78.4               h6a678d5_0  \n",
      "greenlet                  3.0.3                    pypi_0    pypi\n",
      "humanfriendly             10.0                     pypi_0    pypi\n",
      "humanize                  4.10.0                   pypi_0    pypi\n",
      "icu                       68.2                 h9c3ff4c_0    conda-forge\n",
      "idna                      3.4              py39h06a4308_0  \n",
      "importlib-metadata        8.2.0              pyha770c72_0    conda-forge\n",
      "importlib_metadata        8.2.0                hd8ed1ab_0    conda-forge\n",
      "importlib_resources       6.4.0              pyhd8ed1ab_0    conda-forge\n",
      "ipykernel                 6.20.2             pyh210e3f2_0    conda-forge\n",
      "ipython                   8.18.1             pyh707e725_3    conda-forge\n",
      "ipython_genutils          0.2.0              pyhd8ed1ab_1    conda-forge\n",
      "ipywidgets                8.0.6              pyhd8ed1ab_0    conda-forge\n",
      "isal                      1.7.0                    pypi_0    pypi\n",
      "jedi                      0.19.1             pyhd8ed1ab_0    conda-forge\n",
      "jinja2                    3.1.4              pyhd8ed1ab_0    conda-forge\n",
      "json5                     0.9.25             pyhd8ed1ab_0    conda-forge\n",
      "jsonpatch                 1.32               pyhd3eb1b0_0  \n",
      "jsonpointer               2.1                pyhd3eb1b0_0  \n",
      "jsonschema                4.17.3             pyhd8ed1ab_0    conda-forge\n",
      "jupyter_client            7.3.4              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_core              5.7.2            py39hf3d152e_0    conda-forge\n",
      "jupyter_events            0.6.3              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_server            1.24.0             pyhd8ed1ab_0    conda-forge\n",
      "jupyter_server_fileid     0.9.2              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_server_ydoc       0.6.1              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_ydoc              0.2.4              pyhd8ed1ab_0    conda-forge\n",
      "jupyterlab                3.6.1              pyhd8ed1ab_0    conda-forge\n",
      "jupyterlab-dx-extension   0.1.0                    pypi_0    pypi\n",
      "jupyterlab_pygments       0.3.0              pyhd8ed1ab_0    conda-forge\n",
      "jupyterlab_server         2.24.0             pyhd8ed1ab_0    conda-forge\n",
      "jupyterlab_widgets        3.0.11             pyhd8ed1ab_0    conda-forge\n",
      "kaleido                   0.2.1                    pypi_0    pypi\n",
      "kiwisolver                1.4.5                    pypi_0    pypi\n",
      "ld_impl_linux-64          2.38                 h1181459_1  \n",
      "libffi                    3.4.2                h6a678d5_6  \n",
      "libgcc-ng                 11.2.0               h1234567_1  \n",
      "libglib                   2.78.4               hdc74915_0  \n",
      "libgomp                   11.2.0               h1234567_1  \n",
      "libiconv                  1.16                 h5eee18b_3  \n",
      "libpng                    1.6.39               h5eee18b_0  \n",
      "libsodium                 1.0.18               h36c2ea0_1    conda-forge\n",
      "libstdcxx-ng              11.2.0               h1234567_1  \n",
      "libuv                     1.40.0               h7f98852_0    conda-forge\n",
      "libxcb                    1.15                 h7f8727e_0  \n",
      "markdown                  3.7                      pypi_0    pypi\n",
      "markdown-it-py            3.0.0                    pypi_0    pypi\n",
      "markupsafe                2.1.1            py39hb9d737c_1    conda-forge\n",
      "matplotlib                3.6.3                    pypi_0    pypi\n",
      "matplotlib-inline         0.1.7              pyhd8ed1ab_0    conda-forge\n",
      "mdurl                     0.1.2                    pypi_0    pypi\n",
      "mistune                   3.0.2              pyhd8ed1ab_0    conda-forge\n",
      "multiqc                   1.24.1                   pypi_0    pypi\n",
      "nbclassic                 1.1.0              pyhd8ed1ab_0    conda-forge\n",
      "nbclient                  0.10.0             pyhd8ed1ab_0    conda-forge\n",
      "nbconvert-core            7.16.4             pyhd8ed1ab_1    conda-forge\n",
      "nbformat                  5.10.4             pyhd8ed1ab_0    conda-forge\n",
      "ncurses                   6.4                  h6a678d5_0  \n",
      "nest-asyncio              1.6.0              pyhd8ed1ab_0    conda-forge\n",
      "networkx                  2.8.8                    pypi_0    pypi\n",
      "nodejs                    16.13.1              hb931c9a_0  \n",
      "notebook                  6.5.4              pyha770c72_0    conda-forge\n",
      "notebook-shim             0.2.4              pyhd8ed1ab_0    conda-forge\n",
      "numpy                     1.23.5                   pypi_0    pypi\n",
      "openjdk                   11.0.13              h87a67e3_0  \n",
      "openssl                   1.1.1w               h7f8727e_0  \n",
      "packaging                 23.0             py39h06a4308_0  \n",
      "pandas                    1.5.3                    pypi_0    pypi\n",
      "pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge\n",
      "papermill                 2.4.0                    pypi_0    pypi\n",
      "parso                     0.8.4              pyhd8ed1ab_0    conda-forge\n",
      "pcre2                     10.42                hebb0a14_1  \n",
      "perl                      5.34.0               h5eee18b_2  \n",
      "pexpect                   4.9.0              pyhd8ed1ab_0    conda-forge\n",
      "pickleshare               0.7.5                   py_1003    conda-forge\n",
      "pigz                      2.9rc3                   pypi_0    pypi\n",
      "pillow                    10.4.0                   pypi_0    pypi\n",
      "pip                       23.0.1           py39h06a4308_0  \n",
      "pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge\n",
      "platformdirs              4.2.2              pyhd8ed1ab_0    conda-forge\n",
      "plotly                    5.24.0                   pypi_0    pypi\n",
      "pluggy                    1.0.0            py39h06a4308_1  \n",
      "prometheus_client         0.20.0             pyhd8ed1ab_0    conda-forge\n",
      "prompt-toolkit            3.0.47             pyha770c72_0    conda-forge\n",
      "psutil                    6.0.0                    pypi_0    pypi\n",
      "ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge\n",
      "pure-sasl                 0.6.2                    pypi_0    pypi\n",
      "pure_eval                 0.2.3              pyhd8ed1ab_0    conda-forge\n",
      "py4j                      0.10.9.5                 pypi_0    pypi\n",
      "pyaml-env                 1.2.1                    pypi_0    pypi\n",
      "pycosat                   0.6.4            py39h5eee18b_0  \n",
      "pycparser                 2.21               pyhd3eb1b0_0  \n",
      "pydantic                  2.8.2                    pypi_0    pypi\n",
      "pydantic-core             2.20.1                   pypi_0    pypi\n",
      "pygments                  2.18.0             pyhd8ed1ab_0    conda-forge\n",
      "pyhive                    0.7.0                    pypi_0    pypi\n",
      "pyopenssl                 23.0.0           py39h06a4308_0  \n",
      "pypandoc                  1.13                     pypi_0    pypi\n",
      "pyparsing                 3.1.2                    pypi_0    pypi\n",
      "pyrsistent                0.18.1           py39hb9d737c_1    conda-forge\n",
      "pysocks                   1.7.1            py39h06a4308_0  \n",
      "pyspark                   3.2.3                    pypi_0    pypi\n",
      "python                    3.9.16               h7a1cb2a_2  \n",
      "python-dateutil           2.9.0              pyhd8ed1ab_0    conda-forge\n",
      "python-fastjsonschema     2.20.0             pyhd8ed1ab_0    conda-forge\n",
      "python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge\n",
      "python_abi                3.9                      2_cp39    conda-forge\n",
      "pytz                      2024.1             pyhd8ed1ab_0    conda-forge\n",
      "pyyaml                    5.4.1                    pypi_0    pypi\n",
      "pyzmq                     23.0.0           py39headdf64_0    conda-forge\n",
      "readline                  8.2                  h5eee18b_0  \n",
      "requests                  2.32.0                   pypi_0    pypi\n",
      "rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge\n",
      "rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge\n",
      "rich                      13.8.0                   pypi_0    pypi\n",
      "rich-click                1.8.3                    pypi_0    pypi\n",
      "ruamel.yaml               0.17.21          py39h5eee18b_0  \n",
      "ruamel.yaml.clib          0.2.6            py39h5eee18b_1  \n",
      "sasl                      0.3.1                    pypi_0    pypi\n",
      "scipy                     1.9.3                    pypi_0    pypi\n",
      "seaborn                   0.12.2                   pypi_0    pypi\n",
      "send2trash                1.8.3              pyh0d859eb_0    conda-forge\n",
      "setuptools                65.6.3           py39h06a4308_0  \n",
      "six                       1.16.0             pyhd3eb1b0_1  \n",
      "sniffio                   1.3.1              pyhd8ed1ab_0    conda-forge\n",
      "soupsieve                 2.5                pyhd8ed1ab_1    conda-forge\n",
      "spectra                   0.0.11                   pypi_0    pypi\n",
      "sqlalchemy                1.4.49                   pypi_0    pypi\n",
      "sqlite                    3.41.1               h5eee18b_0  \n",
      "sqlparse                  0.5.1                    pypi_0    pypi\n",
      "stack_data                0.6.2              pyhd8ed1ab_0    conda-forge\n",
      "tenacity                  9.0.0                    pypi_0    pypi\n",
      "terminado                 0.18.1             pyh0d859eb_0    conda-forge\n",
      "textwrap3                 0.9.2                    pypi_0    pypi\n",
      "thrift                    0.20.0                   pypi_0    pypi\n",
      "thrift-sasl               0.4.3                    pypi_0    pypi\n",
      "tinycss2                  1.3.0              pyhd8ed1ab_0    conda-forge\n",
      "tk                        8.6.12               h1ccaba5_0  \n",
      "tomli                     2.0.1              pyhd8ed1ab_0    conda-forge\n",
      "toolz                     0.12.0           py39h06a4308_0  \n",
      "tornado                   6.1              py39hb9d737c_3    conda-forge\n",
      "tqdm                      4.65.0           py39hb070fc8_0  \n",
      "traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge\n",
      "typeguard                 4.3.0                    pypi_0    pypi\n",
      "typing                    3.7.4.1                  pypi_0    pypi\n",
      "typing-extensions         4.12.2               hd8ed1ab_0    conda-forge\n",
      "typing_extensions         4.12.2             pyha770c72_0    conda-forge\n",
      "tzdata                    2023c                h04d1e81_0  \n",
      "urllib3                   1.26.15          py39h06a4308_0  \n",
      "wcwidth                   0.2.13             pyhd8ed1ab_0    conda-forge\n",
      "webencodings              0.5.1              pyhd8ed1ab_2    conda-forge\n",
      "websocket-client          1.7.0                    pypi_0    pypi\n",
      "wheel                     0.38.4           py39h06a4308_0  \n",
      "widgetsnbextension        4.0.11             pyhd8ed1ab_0    conda-forge\n",
      "xopen                     2.0.2                    pypi_0    pypi\n",
      "xz                        5.2.10               h5eee18b_1  \n",
      "y-py                      0.5.9            py39h52d8a92_0  \n",
      "yaml                      0.2.5                h7f98852_2    conda-forge\n",
      "ypy-websocket             0.8.2              pyhd8ed1ab_0    conda-forge\n",
      "zeromq                    4.3.4                h9c3ff4c_1    conda-forge\n",
      "zipp                      3.19.2             pyhd8ed1ab_0    conda-forge\n",
      "zlib                      1.2.13               h5eee18b_0  \n",
      "zlib-ng                   0.5.0                    pypi_0    pypi\n",
      "zstandard                 0.19.0           py39h5eee18b_0  \n"
     ]
    }
   ],
   "source": [
    "!conda info\n",
    "!conda list"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "de953afa-b703-49bb-b710-7c025db7c495",
   "metadata": {},
   "source": [
    "## fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "f3ef0c26-6e74-46cd-83fd-472f22704f2f",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "            FastQC - A high throughput sequence QC analysis tool\n",
      "\n",
      "SYNOPSIS\n",
      "\n",
      "\tfastqc seqfile1 seqfile2 .. seqfileN\n",
      "\n",
      "    fastqc [-o output dir] [--(no)extract] [-f fastq|bam|sam] \n",
      "           [-c contaminant file] seqfile1 .. seqfileN\n",
      "\n",
      "DESCRIPTION\n",
      "\n",
      "    FastQC reads a set of sequence files and produces from each one a quality\n",
      "    control report consisting of a number of different modules, each one of \n",
      "    which will help to identify a different potential type of problem in your\n",
      "    data.\n",
      "    \n",
      "    If no files to process are specified on the command line then the program\n",
      "    will start as an interactive graphical application.  If files are provided\n",
      "    on the command line then the program will run with no user interaction\n",
      "    required.  In this mode it is suitable for inclusion into a standardised\n",
      "    analysis pipeline.\n",
      "    \n",
      "    The options for the program as as follows:\n",
      "    \n",
      "    -h --help       Print this help file and exit\n",
      "    \n",
      "    -v --version    Print the version of the program and exit\n",
      "    \n",
      "    -o --outdir     Create all output files in the specified output directory.\n",
      "                    Please note that this directory must exist as the program\n",
      "                    will not create it.  If this option is not set then the \n",
      "                    output file for each sequence file is created in the same\n",
      "                    directory as the sequence file which was processed.\n",
      "                    \n",
      "    --casava        Files come from raw casava output. Files in the same sample\n",
      "                    group (differing only by the group number) will be analysed\n",
      "                    as a set rather than individually. Sequences with the filter\n",
      "                    flag set in the header will be excluded from the analysis.\n",
      "                    Files must have the same names given to them by casava\n",
      "                    (including being gzipped and ending with .gz) otherwise they\n",
      "                    won't be grouped together correctly.\n",
      "                    \n",
      "    --nano          Files come from nanopore sequences and are in fast5 format. In\n",
      "                    this mode you can pass in directories to process and the program\n",
      "                    will take in all fast5 files within those directories and produce\n",
      "                    a single output file from the sequences found in all files.                    \n",
      "                    \n",
      "    --nofilter      If running with --casava then don't remove read flagged by\n",
      "                    casava as poor quality when performing the QC analysis.\n",
      "                   \n",
      "    --extract       If set then the zipped output file will be uncompressed in\n",
      "                    the same directory after it has been created.  By default\n",
      "                    this option will be set if fastqc is run in non-interactive\n",
      "                    mode.\n",
      "                    \n",
      "    -j --java       Provides the full path to the java binary you want to use to\n",
      "                    launch fastqc. If not supplied then java is assumed to be in\n",
      "                    your path.\n",
      "                   \n",
      "    --noextract     Do not uncompress the output file after creating it.  You\n",
      "                    should set this option if you do not wish to uncompress\n",
      "                    the output when running in non-interactive mode.\n",
      "                    \n",
      "    --nogroup       Disable grouping of bases for reads >50bp. All reports will\n",
      "                    show data for every base in the read.  WARNING: Using this\n",
      "                    option will cause fastqc to crash and burn if you use it on\n",
      "                    really long reads, and your plots may end up a ridiculous size.\n",
      "                    You have been warned!\n",
      "                    \n",
      "    --min_length    Sets an artificial lower limit on the length of the sequence\n",
      "                    to be shown in the report.  As long as you set this to a value\n",
      "                    greater or equal to your longest read length then this will be\n",
      "                    the sequence length used to create your read groups.  This can\n",
      "                    be useful for making directly comaparable statistics from \n",
      "                    datasets with somewhat variable read lengths.\n",
      "                    \n",
      "    -f --format     Bypasses the normal sequence file format detection and\n",
      "                    forces the program to use the specified format.  Valid\n",
      "                    formats are bam,sam,bam_mapped,sam_mapped and fastq\n",
      "                    \n",
      "    -t --threads    Specifies the number of files which can be processed\n",
      "                    simultaneously.  Each thread will be allocated 250MB of\n",
      "                    memory so you shouldn't run more threads than your\n",
      "                    available memory will cope with, and not more than\n",
      "                    6 threads on a 32 bit machine\n",
      "                  \n",
      "    -c              Specifies a non-default file which contains the list of\n",
      "    --contaminants  contaminants to screen overrepresented sequences against.\n",
      "                    The file must contain sets of named contaminants in the\n",
      "                    form name[tab]sequence.  Lines prefixed with a hash will\n",
      "                    be ignored.\n",
      "\n",
      "    -a              Specifies a non-default file which contains the list of\n",
      "    --adapters      adapter sequences which will be explicity searched against\n",
      "                    the library. The file must contain sets of named adapters\n",
      "                    in the form name[tab]sequence.  Lines prefixed with a hash\n",
      "                    will be ignored.\n",
      "                    \n",
      "    -l              Specifies a non-default file which contains a set of criteria\n",
      "    --limits        which will be used to determine the warn/error limits for the\n",
      "                    various modules.  This file can also be used to selectively \n",
      "                    remove some modules from the output all together.  The format\n",
      "                    needs to mirror the default limits.txt file found in the\n",
      "                    Configuration folder.\n",
      "                    \n",
      "   -k --kmers       Specifies the length of Kmer to look for in the Kmer content\n",
      "                    module. Specified Kmer length must be between 2 and 10. Default\n",
      "                    length is 7 if not specified.\n",
      "                    \n",
      "   -q --quiet       Supress all progress messages on stdout and only report errors.\n",
      "   \n",
      "   -d --dir         Selects a directory to be used for temporary files written when\n",
      "                    generating report images. Defaults to system temp directory if\n",
      "                    not specified.\n",
      "                    \n",
      "BUGS\n",
      "\n",
      "    Any bugs in fastqc should be reported either to simon.andrews@babraham.ac.uk\n",
      "    or in www.bioinformatics.babraham.ac.uk/bugzilla/\n",
      "                   \n",
      "    \n"
     ]
    }
   ],
   "source": [
    "!fastqc --help"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "bdc58f67-b049-449c-8da9-159c2f76f2b2",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#!/bin/bash\n",
      "\n",
      "# Define the base directory and output directory\n",
      "BASE_DIR=\"20240827_abundant_emu\"\n",
      "OUTPUT_DIR=\"01_fastqc\"\n",
      "\n",
      "# Create the output directory if it doesn't exist\n",
      "mkdir -p $OUTPUT_DIR\n",
      "\n",
      "# Find all .fastq.gz files and run FastQC on them\n",
      "find $BASE_DIR -type f -name \"*.fastq.gz\" | while read file; do\n",
      "    echo \"Running FastQC on $file\"\n",
      "    fastqc \"$file\" --outdir=\"$OUTPUT_DIR\"\n",
      "done\n"
     ]
    }
   ],
   "source": [
    "!cat run_fastqc.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "b52a9862-0a8b-4db2-a59e-a32e267bfb49",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "!chmod +x run_fastqc.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "55b826b7-e3f5-4cd1-93f4-a47191f75f1f",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Running FastQC on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Started analysis of SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 5% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 10% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 15% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 20% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 25% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 30% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 35% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 40% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 45% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 50% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 55% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 60% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 65% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 70% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 75% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 80% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 85% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 90% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Approx 95% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Analysis complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!./run_fastqc.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4f59fdfa-1212-4e6f-9ff9-df35fd0c1b56",
   "metadata": {},
   "source": [
    "## trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "8419dcac-e83c-4745-9384-478e43a30ba7",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Multicore support not enabled. Proceeding with single-core trimming.\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "single-core operation.\n",
      "Proceeding with 'gzip' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "\n",
      "Please provide the filename(s) of one or more FastQ file(s) to launch Trim Galore!\n",
      "\n",
      "USAGE:  'trim_galore [options] <filename(s)>'    or    'trim_galore --help'    for more options\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!TrimGalore-0.6.10/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "35a7f693-8772-4999-9884-e3b74036afb7",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " USAGE:\n",
      "\n",
      "trim_galore [options] <filename(s)>\n",
      "\n",
      "\n",
      "-h/--help               Print this help message and exits.\n",
      "\n",
      "-v/--version            Print the version information and exits.\n",
      "\n",
      "-q/--quality <INT>      Trim low-quality ends from reads in addition to adapter removal. For\n",
      "                        RRBS samples, quality trimming will be performed first, and adapter\n",
      "                        trimming is carried in a second round. Other files are quality and adapter\n",
      "                        trimmed in a single pass. The algorithm is the same as the one used by BWA\n",
      "                        (Subtract INT from all qualities; compute partial sums from all indices\n",
      "                        to the end of the sequence; cut sequence at the index at which the sum is\n",
      "                        minimal). Default Phred score: 20.\n",
      "\n",
      "--phred33               Instructs Cutadapt to use ASCII+33 quality scores as Phred scores\n",
      "                        (Sanger/Illumina 1.9+ encoding) for quality trimming. Default: ON.\n",
      "\n",
      "--phred64               Instructs Cutadapt to use ASCII+64 quality scores as Phred scores\n",
      "                        (Illumina 1.5 encoding) for quality trimming.\n",
      "\n",
      "--fastqc                Run FastQC in the default mode on the FastQ file once trimming is complete.\n",
      "\n",
      "--fastqc_args \"<ARGS>\"  Passes extra arguments to FastQC. If more than one argument is to be passed\n",
      "                        to FastQC they must be in the form \"arg1 arg2 etc.\". An example would be:\n",
      "                        --fastqc_args \"--nogroup --outdir /home/\". Passing extra arguments will\n",
      "                        automatically invoke FastQC, so --fastqc does not have to be specified\n",
      "                        separately.\n",
      "\n",
      "-a/--adapter <STRING>   Adapter sequence to be trimmed. If not specified explicitly, Trim Galore will\n",
      "                        try to auto-detect whether the Illumina universal, Nextera transposase or Illumina\n",
      "                        small RNA adapter sequence was used. Also see '--illumina', '--nextera' and\n",
      "                        '--small_rna'. If no adapter can be detected within the first 1 million sequences\n",
      "                        of the first file specified or if there is a tie between several adapter sequences,\n",
      "                        Trim Galore defaults to '--illumina' (as long as the Illumina adapter was one of the\n",
      "                        options, else '--nextera' is the default). A single base\n",
      "                        may also be given as e.g. -a A{10}, to be expanded to -a AAAAAAAAAA.\n",
      "\n",
      "                        At a special request, multiple adapters can also be specified like so:\n",
      "                        -a  \" AGCTCCCG -a TTTCATTATAT -a TTTATTCGGATTTAT\"\n",
      "                        -a2 \" AGCTAGCG -a TCTCTTATAT -a TTTCGGATTTAT\", or so:\n",
      "                        -a \"file:../multiple_adapters.fa\"\n",
      "                        -a2 \"file:../different_adapters.fa\"\n",
      "                        Potentially in conjucntion with the parameter \"-n 3\" to trim all adapters. Please note\n",
      "                        that this is NOT needed for standard trimming! \n",
      "                        More Information here: https://github.com/FelixKrueger/TrimGalore/issues/86\n",
      "\n",
      "-a2/--adapter2 <STRING> Optional adapter sequence to be trimmed off read 2 of paired-end files. This\n",
      "                        option requires '--paired' to be specified as well. If the libraries to be trimmed\n",
      "                        are smallRNA then a2 will be set to the Illumina small RNA 5' adapter automatically\n",
      "                        (GATCGTCGGACT). A single base may also be given as e.g. -a2 A{10}, to be expanded\n",
      "                        to -a2 AAAAAAAAAA.\n",
      "\n",
      "--illumina              Adapter sequence to be trimmed is the first 13bp of the Illumina universal adapter\n",
      "                        'AGATCGGAAGAGC' instead of the default auto-detection of adapter sequence.\n",
      "\n",
      "--stranded_illumina     Adapter sequence to be trimmed is the first 13bp of the Illumina stranded mRNA or Total\n",
      "                        RNA adapter 'ACTGTCTCTTATA' instead of the default auto-detection of adapter sequence. \n",
      "                        Note that this sequence resembles the Nextera sequence with an additional A from A-tailing.\n",
      "                        Please also see https://github.com/FelixKrueger/TrimGalore/issues/127 or \n",
      "                        https://support.illumina.com/bulletins/2020/06/trimming-t-overhang-options-for-the-illumina-rna-library-prep-wo.html\n",
      "                        for further information. This sequence is currently NOT included in the adapter auto-detection.\n",
      "\n",
      "--nextera               Adapter sequence to be trimmed is the first 12bp of the Nextera adapter\n",
      "                        'CTGTCTCTTATA' instead of the default auto-detection of adapter sequence.\n",
      "\n",
      "--small_rna             Adapter sequence to be trimmed is the first 12bp of the Illumina Small RNA 3' Adapter\n",
      "                        'TGGAATTCTCGG' instead of the default auto-detection of adapter sequence. Selecting\n",
      "                        to trim smallRNA adapters will also lower the --length value to 18bp. If the smallRNA\n",
      "                        libraries are paired-end then a2 will be set to the Illumina small RNA 5' adapter\n",
      "                        automatically (GATCGTCGGACT) unless -a 2 had been defined explicitly.\n",
      "\n",
      "--consider_already_trimmed <INT>     During adapter auto-detection, the limit set by <INT> allows the user to \n",
      "                        set a threshold up to which the file is considered already adapter-trimmed. If no adapter\n",
      "                        sequence exceeds this threshold, no additional adapter trimming will be performed (technically,\n",
      "                        the adapter is set to '-a X'). Quality trimming is still performed as usual.\n",
      "                        Default: NOT SELECTED (i.e. normal auto-detection precedence rules apply).                     \n",
      "\n",
      "--max_length <INT>      Discard reads that are longer than <INT> bp after trimming. This is only advised for\n",
      "                        smallRNA sequencing to remove non-small RNA sequences.\n",
      "\n",
      "\n",
      "--stringency <INT>      Overlap with adapter sequence required to trim a sequence. Defaults to a\n",
      "                        very stringent setting of 1, i.e. even a single bp of overlapping sequence\n",
      "                        will be trimmed off from the 3' end of any read.\n",
      "\n",
      "-e <ERROR RATE>         Maximum allowed error rate (no. of errors divided by the length of the matching\n",
      "                        region) (default: 0.1)\n",
      "\n",
      "--gzip                  Compress the output file with GZIP. If the input files are GZIP-compressed\n",
      "                        the output files will automatically be GZIP compressed as well. As of v0.2.8 the\n",
      "                        compression will take place on the fly.\n",
      "\n",
      "--dont_gzip             Output files won't be compressed with GZIP. This option overrides --gzip.\n",
      "\n",
      "--length <INT>          Discard reads that became shorter than length INT because of either\n",
      "                        quality or adapter trimming. A value of '0' effectively disables\n",
      "                        this behaviour. Default: 20 bp.\n",
      "\n",
      "                        For paired-end files, both reads of a read-pair need to be longer than\n",
      "                        <INT> bp to be printed out to validated paired-end files (see option --paired).\n",
      "                        If only one read became too short there is the possibility of keeping such\n",
      "                        unpaired single-end reads (see --retain_unpaired). Default pair-cutoff: 20 bp.\n",
      "\n",
      "--max_n COUNT           The total number of Ns a read may contain before it will be removed altogether.\n",
      "                        In a paired-end setting, either read exceeding this limit will result in the entire\n",
      "                        pair being removed from the trimmed output files. If COUNT is a number between 0 and 1,\n",
      "                        it is interpreted as a fraction of the read length.\n",
      "\n",
      "--trim-n                Removes Ns from either side of the read. This option does currently not work in RRBS mode.\n",
      "\n",
      "-o/--output_dir <DIR>   If specified all output will be written to this directory instead of the current\n",
      "                        directory. If the directory doesn't exist it will be created for you.\n",
      "\n",
      "--no_report_file        If specified no report file will be generated.\n",
      "\n",
      "--suppress_warn         If specified any output to STDOUT or STDERR will be suppressed.\n",
      "\n",
      "--clip_R1 <int>         Instructs Trim Galore to remove <int> bp from the 5' end of read 1 (or single-end\n",
      "                        reads). This may be useful if the qualities were very poor, or if there is some\n",
      "                        sort of unwanted bias at the 5' end. Default: OFF.\n",
      "\n",
      "--clip_R2 <int>         Instructs Trim Galore to remove <int> bp from the 5' end of read 2 (paired-end reads\n",
      "                        only). This may be useful if the qualities were very poor, or if there is some sort\n",
      "                        of unwanted bias at the 5' end. For paired-end BS-Seq, it is recommended to remove\n",
      "                        the first few bp because the end-repair reaction may introduce a bias towards low\n",
      "                        methylation. Please refer to the M-bias plot section in the Bismark User Guide for\n",
      "                        some examples. Default: OFF.\n",
      "\n",
      "--three_prime_clip_R1 <int>     Instructs Trim Galore to remove <int> bp from the 3' end of read 1 (or single-end\n",
      "                        reads) AFTER adapter/quality trimming has been performed. This may remove some unwanted\n",
      "                        bias from the 3' end that is not directly related to adapter sequence or basecall quality.\n",
      "                        Default: OFF.\n",
      "\n",
      "--three_prime_clip_R2 <int>     Instructs Trim Galore to remove <int> bp from the 3' end of read 2 AFTER\n",
      "                        adapter/quality trimming has been performed. This may remove some unwanted bias from\n",
      "                        the 3' end that is not directly related to adapter sequence or basecall quality.\n",
      "                        Default: OFF.\n",
      "\n",
      "--2colour/--nextseq INT This enables the option '--nextseq-trim=3'CUTOFF' within Cutadapt, which will set a quality\n",
      "                        cutoff (that is normally given with -q instead), but qualities of G bases are ignored.\n",
      "                        This trimming is in common for the NextSeq- and NovaSeq-platforms, where basecalls without\n",
      "                        any signal are called as high-quality G bases. This is mutually exlusive with '-q INT'.\n",
      "\n",
      "\n",
      "--path_to_cutadapt </path/to/cutadapt>     You may use this option to specify a path to the Cutadapt executable,\n",
      "                        e.g. /my/home/cutadapt-1.7.1/bin/cutadapt. Else it is assumed that Cutadapt is in\n",
      "                        the PATH.\n",
      "\n",
      "--basename <PREFERRED_NAME>\tUse PREFERRED_NAME as the basename for output files, instead of deriving the filenames from\n",
      "                        the input files. Single-end data would be called PREFERRED_NAME_trimmed.fq(.gz), or\n",
      "                        PREFERRED_NAME_val_1.fq(.gz) and PREFERRED_NAME_val_2.fq(.gz) for paired-end data. --basename\n",
      "                        only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.\n",
      "\n",
      "-j/--cores INT          Number of cores to be used for trimming [default: 1]. For Cutadapt to work with multiple cores, it\n",
      "                        requires Python 3 as well as parallel gzip (pigz) installed on the system. Trim Galore attempts to detect\n",
      "                        the version of Python used by calling Cutadapt. If Python 2 is detected, --cores is set to 1. If the Python\n",
      "                        version cannot be detected, Python 3 is assumed and we let Cutadapt handle potential issues itself.\n",
      "                        \n",
      "                        If pigz cannot be detected on your system, Trim Galore reverts to using gzip compression. Please note\n",
      "                        that gzip compression will slow down multi-core processes so much that it is hardly worthwhile, please \n",
      "                        see: https://github.com/FelixKrueger/TrimGalore/issues/16#issuecomment-458557103 for more info).\n",
      "\t\t\t\t\t\t\n",
      "                        Actual core usage: It should be mentioned that the actual number of cores used is a little convoluted.\n",
      "                        Assuming that Python 3 is used and pigz is installed, --cores 2 would use 2 cores to read the input\n",
      "                        (probably not at a high usage though), 2 cores to write to the output (at moderately high usage), and \n",
      "                        2 cores for Cutadapt itself + 2 additional cores for Cutadapt (not sure what they are used for) + 1 core\n",
      "                        for Trim Galore itself. So this can be up to 9 cores, even though most of them won't be used at 100% for\n",
      "                        most of the time. Paired-end processing uses twice as many cores for the validation (= writing out) step.\n",
      "                        --cores 4 would then be: 4 (read) + 4 (write) + 4 (Cutadapt) + 2 (extra Cutadapt) +\t1 (Trim Galore) = 15.\n",
      "\n",
      "                        It seems that --cores 4 could be a sweet spot, anything above has diminishing returns.\n",
      "\t\t\t\n",
      "\n",
      "\n",
      "SPECIFIC TRIMMING - without adapter/quality trimming\n",
      "\n",
      "--hardtrim5 <int>       Instead of performing adapter-/quality trimming, this option will simply hard-trim sequences\n",
      "                        to <int> bp at the 5'-end. Once hard-trimming of files is complete, Trim Galore will exit.\n",
      "                        Hard-trimmed output files will end in .<int>_5prime.fq(.gz). Here is an example:\n",
      "\n",
      "                        before:         CCTAAGGAAACAAGTACACTCCACACATGCATAAAGGAAATCAAATGTTATTTTTAAGAAAATGGAAAAT\n",
      "                        --hardtrim5 20: CCTAAGGAAACAAGTACACT\n",
      "\n",
      "--hardtrim3 <int>       Instead of performing adapter-/quality trimming, this option will simply hard-trim sequences\n",
      "                        to <int> bp at the 3'-end. Once hard-trimming of files is complete, Trim Galore will exit.\n",
      "                        Hard-trimmed output files will end in .<int>_3prime.fq(.gz). Here is an example:\n",
      "\n",
      "                        before:         CCTAAGGAAACAAGTACACTCCACACATGCATAAAGGAAATCAAATGTTATTTTTAAGAAAATGGAAAAT\n",
      "                        --hardtrim3 20:                                                   TTTTTAAGAAAATGGAAAAT\n",
      "\n",
      "--clock                 In this mode, reads are trimmed in a specific way that is currently used for the Mouse\n",
      "                        Epigenetic Clock (see here: Multi-tissue DNA methylation age predictor in mouse, Stubbs et al.,\n",
      "                        Genome Biology, 2017 18:68 https://doi.org/10.1186/s13059-017-1203-5). Following this, Trim Galore\n",
      "                        will exit.\n",
      "\n",
      "                        In it's current implementation, the dual-UMI RRBS reads come in the following format:\n",
      "\n",
      "                        Read 1  5' UUUUUUUU CAGTA FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF TACTG UUUUUUUU 3'\n",
      "                        Read 2  3' UUUUUUUU GTCAT FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF ATGAC UUUUUUUU 5'\n",
      "\n",
      "                        Where UUUUUUUU is a random 8-mer unique molecular identifier (UMI), CAGTA is a constant region,\n",
      "                        and FFFFFFF... is the actual RRBS-Fragment to be sequenced. The UMIs for Read 1 (R1) and\n",
      "                        Read 2 (R2), as well as the fixed sequences (F1 or F2), are written into the read ID and\n",
      "                        removed from the actual sequence. Here is an example:\n",
      "\n",
      "                        R1: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 1:N:0: CGATGTTT\n",
      "                            ATCTAGTTCAGTACGGTGTTTTCGAATTAGAAAAATATGTATAGAGGAAATAGATATAAAGGCGTATTCGTTATTG\n",
      "                        R2: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 3:N:0: CGATGTTT\n",
      "                            CAATTTTGCAGTACAAAAATAATACCTCCTCTATTTATCCAAAATCACAAAAAACCACCCACTTAACTTTCCCTAA\n",
      "\n",
      "                        R1: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 1:N:0: CGATGTTT:R1:ATCTAGTT:R2:CAATTTTG:F1:CAGT:F2:CAGT\n",
      "                                         CGGTGTTTTCGAATTAGAAAAATATGTATAGAGGAAATAGATATAAAGGCGTATTCGTTATTG\n",
      "                        R2: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 3:N:0: CGATGTTT:R1:ATCTAGTT:R2:CAATTTTG:F1:CAGT:F2:CAGT\n",
      "                                         CAAAAATAATACCTCCTCTATTTATCCAAAATCACAAAAAACCACCCACTTAACTTTCCCTAA\n",
      "\n",
      "                        Following clock trimming, the resulting files (.clock_UMI.R1.fq(.gz) and .clock_UMI.R2.fq(.gz))\n",
      "                        should be adapter- and quality trimmed with Trim Galore as usual. In addition, reads need to be trimmed\n",
      "                        by 15bp from their 3' end to get rid of potential UMI and fixed sequences. The command is:\n",
      "\n",
      "                        trim_galore --paired --three_prime_clip_R1 15 --three_prime_clip_R2 15 *.clock_UMI.R1.fq.gz *.clock_UMI.R2.fq.gz\n",
      "\n",
      "                        Following this, reads should be aligned with Bismark and deduplicated with UmiBam\n",
      "                        in '--dual_index' mode (see here: https://github.com/FelixKrueger/Umi-Grinder). UmiBam recognises\n",
      "                        the UMIs within this pattern: R1:(ATCTAGTT):R2:(CAATTTTG): as (UMI R1) and (UMI R2).\n",
      "\n",
      "--polyA                 This is a new, still experimental, trimming mode to identify and remove poly-A tails from sequences.\n",
      "                        When --polyA is selected, Trim Galore attempts to identify from the first supplied sample whether\n",
      "                        sequences contain more often a stretch of either 'AAAAAAAAAA' or 'TTTTTTTTTT'. This determines\n",
      "                        if Read 1 of a paired-end end file, or single-end files, are trimmed for PolyA or PolyT. In case of\n",
      "                        paired-end sequencing, Read2 is trimmed for the complementary base from the start of the reads. The\n",
      "                        auto-detection uses a default of A{20} for Read1 (3'-end trimming) and T{150} for Read2 (5'-end trimming).\n",
      "                        These values may be changed manually using the options -a and -a2.\n",
      "\n",
      "                        In addition to trimming the sequences, white spaces are replaced with _ and it records in the read ID\n",
      "                        how many bases were trimmed so it can later be used to identify PolyA trimmed sequences. This is currently done\n",
      "                        by writing tags to both the start (\"32:A:\") and end (\"_PolyA:32\") of the reads in the following example:\n",
      "\n",
      "                        @READ-ID:1:1102:22039:36996 1:N:0:CCTAATCC\n",
      "                        GCCTAAGGAAACAAGTACACTCCACACATGCATAAAGGAAATCAAATGTTATTTTTAAGAAAATGGAAAATAAAAACTTTATAAACACCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\n",
      "\n",
      "                        @32:A:READ-ID:1:1102:22039:36996_1:N:0:CCTAATCC_PolyA:32\n",
      "                        GCCTAAGGAAACAAGTACACTCCACACATGCATAAAGGAAATCAAATGTTATTTTTAAGAAAATGGAAAATAAAAACTTTATAAACACC\n",
      "\n",
      "                        PLEASE NOTE: The poly-A trimming mode expects that sequences were both adapter and quality trimmed\n",
      "                        before looking for Poly-A tails, and it is the user's responsibility to carry out an initial round of\n",
      "                        trimming. The following sequence:\n",
      " \n",
      "                        1) trim_galore file.fastq.gz\n",
      "                        2) trim_galore --polyA file_trimmed.fq.gz\n",
      "                        3) zcat file_trimmed_trimmed.fq.gz | grep -A 3 PolyA | grep -v ^-- > PolyA_trimmed.fastq\n",
      "\n",
      "                        Will 1) trim qualities and Illumina adapter contamination, 2) find and remove PolyA contamination.\n",
      "                        Finally, if desired, 3) will specifically find PolyA trimmed sequences to a specific FastQ file of your choice.\n",
      "\n",
      "--implicon              This is a special mode of operation for paired-end data, such as required for the IMPLICON method, where a UMI sequence\n",
      "                        is getting transferred from the start of Read 2 to the readID of both reads. Following this, Trim Galore will exit.\n",
      "\n",
      "                        In it's current implementation, the UMI carrying reads come in the following format:\n",
      "\n",
      "                        Read 1  5' FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF 3'\n",
      "                        Read 2  3' UUUUUUUUFFFFFFFFFFFFFFFFFFFFFFFFFFFF 5'\n",
      "\n",
      "                        Where UUUUUUUU is a random 8-mer unique molecular identifier (UMI) and FFFFFFF... is the actual fragment to be\n",
      "                        sequenced. The UMI of Read 2 (R2) is written into the read ID of both reads and removed from the actual sequence.\n",
      "                        Here is an example:\n",
      "\n",
      "                        R1: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 1:N:0: CGATGTTT\n",
      "                            ATCTAGTTCAGTACGGTGTTTTCGAATTAGAAAAATATGTATAGAGGAAATAGATATAAAGGCGTATTCGTTATTG\n",
      "                        R2: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 3:N:0: CGATGTTT\n",
      "                            CAATTTTGCAGTACAAAAATAATACCTCCTCTATTTATCCAAAATCACAAAAAACCACCCACTTAACTTTCCCTAA\n",
      "                        \n",
      "                        After --implicon trimming:\n",
      "                        R1: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 1:N:0: CGATGTTT:CAATTTTG\n",
      "                            ATCTAGTTCAGTACGGTGTTTTCGAATTAGAAAAATATGTATAGAGGAAATAGATATAAAGGCGTATTCGTTATTG\n",
      "                        R2: @HWI-D00436:407:CCAETANXX:1:1101:4105:1905 3:N:0: CGATGTTT:CAATTTTG\n",
      "                                    CAGTACAAAAATAATACCTCCTCTATTTATCCAAAATCACAAAAAACCACCCACTTAACTTTCCCTAA\n",
      "\n",
      "RRBS-specific options (MspI digested material):\n",
      "\n",
      "--rrbs                  Specifies that the input file was an MspI digested RRBS sample (recognition\n",
      "                        site: CCGG). Single-end or Read 1 sequences (paired-end) which were adapter-trimmed\n",
      "                        will have a further 2 bp removed from their 3' end. Sequences which were merely\n",
      "                        trimmed because of poor quality will not be shortened further. Read 2 of paired-end\n",
      "                        libraries will in addition have the first 2 bp removed from the 5' end (by setting\n",
      "                        '--clip_r2 2'). This is to avoid using artificial methylation calls from the filled-in\n",
      "                        cytosine positions close to the 3' MspI site in sequenced fragments.\n",
      "                        This option is not recommended for users of the Tecan Ovation RRBS Methyl-Seq with TrueMethyl\n",
      "                        oxBS 1-16 kit (see below).\n",
      "\n",
      "--non_directional       Selecting this option for non-directional RRBS libraries will screen\n",
      "                        quality-trimmed sequences for 'CAA' or 'CGA' at the start of the read\n",
      "                        and, if found, removes the first two basepairs. Like with the option\n",
      "                        '--rrbs' this avoids using cytosine positions that were filled-in\n",
      "                        during the end-repair step. '--non_directional' requires '--rrbs' to\n",
      "                        be specified as well. Note that this option does not set '--clip_r2 2' in\n",
      "                        paired-end mode.\n",
      "\n",
      "--keep                  Keep the quality trimmed intermediate file. Default: off, which means\n",
      "                        the temporary file is being deleted after adapter trimming. Only has\n",
      "                        an effect for RRBS samples since other FastQ files are not trimmed\n",
      "                        for poor qualities separately.\n",
      "\n",
      "\n",
      "Note for RRBS using the Tecan Ovation RRBS Methyl-Seq with TrueMethyl oxBS 1-16 kit:\n",
      "\n",
      "Owing to the fact that the Tecan Ovation RRBS kit attaches a varying number of nucleotides (0-3) after each MspI\n",
      "site Trim Galore should be run WITHOUT the option --rrbs. This trimming is accomplished in a subsequent\n",
      "diversity trimming step afterwards (see their manual).\n",
      "\n",
      "\n",
      "\n",
      "Note for RRBS using MseI:\n",
      "\n",
      "If your DNA material was digested with MseI (recognition motif: TTAA) instead of MspI it is NOT necessary\n",
      "to specify --rrbs or --non_directional since virtually all reads should start with the sequence\n",
      "'TAA', and this holds true for both directional and non-directional libraries. As the end-repair of 'TAA'\n",
      "restricted sites does not involve any cytosines it does not need to be treated especially. Instead, simply\n",
      "run Trim Galore! in the standard (i.e. non-RRBS) mode.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Paired-end specific options:\n",
      "\n",
      "--paired                This option performs length trimming of quality/adapter/RRBS trimmed reads for\n",
      "                        paired-end files. To pass the validation test, both sequences of a sequence pair\n",
      "                        are required to have a certain minimum length which is governed by the option\n",
      "                        --length (see above). If only one read passes this length threshold the\n",
      "                        other read can be rescued (see option --retain_unpaired). Using this option lets\n",
      "                        you discard too short read pairs without disturbing the sequence-by-sequence order\n",
      "                        of FastQ files which is required by many aligners.\n",
      "\n",
      "                        Trim Galore! expects paired-end files to be supplied in a pairwise fashion, e.g.\n",
      "                        file1_1.fq file1_2.fq SRR2_1.fq.gz SRR2_2.fq.gz ... .\n",
      "\n",
      "\n",
      "--retain_unpaired       If only one of the two paired-end reads became too short, the longer\n",
      "                        read will be written to either '.unpaired_1.fq' or '.unpaired_2.fq'\n",
      "                        output files. The length cutoff for unpaired single-end reads is\n",
      "                        governed by the parameters -r1/--length_1 and -r2/--length_2. Default: OFF.\n",
      "\n",
      "-r1/--length_1 <INT>    Unpaired single-end read length cutoff needed for read 1 to be written to\n",
      "                        '.unpaired_1.fq' output file. These reads may be mapped in single-end mode.\n",
      "                        Default: 35 bp.\n",
      "\n",
      "-r2/--length_2 <INT>    Unpaired single-end read length cutoff needed for read 2 to be written to\n",
      "                        '.unpaired_2.fq' output file. These reads may be mapped in single-end mode.\n",
      "                        Default: 35 bp.\n",
      "\n",
      "Last modified on 02 02 2023.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!TrimGalore-0.6.10/trim_galore --help"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "69764b90-f16c-4f74-9e1f-1788862b5a70",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#!/bin/bash\n",
      "\n",
      "# Define the base directory and output directory\n",
      "BASE_DIR=\"20240827_abundant_emu\"\n",
      "OUTPUT_DIR=\"02_trim_galore\"\n",
      "\n",
      "# Create the output directory if it doesn't exist\n",
      "mkdir -p $OUTPUT_DIR\n",
      "\n",
      "# Find all .fastq.gz files and run Trim Galore on them\n",
      "find $BASE_DIR -type f -name \"*.fastq.gz\" | while read file; do    \n",
      "    # Run Trim Galore\n",
      "    echo \"Running Trim Galore on $file\"\n",
      "    TrimGalore-0.6.10/trim_galore \"$file\" \\\n",
      "    --output_dir \"$OUTPUT_DIR\" \\\n",
      "    --cores 8 \\\n",
      "    --fastqc\n",
      "done\n"
     ]
    }
   ],
   "source": [
    "!cat run_trim_galore.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "01b1133e-5f1f-454f-bda3-5253cd90cd09",
   "metadata": {},
   "outputs": [],
   "source": [
    "!chmod +x run_trim_galore.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "6a52c0cb-8b3e-4cae-862b-c7fe183459b8",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Thu Aug 29 23:26:49 UTC 2024\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 775.467 s (1.143 s/read; 52.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             678,686,021\n",
      "Reads with adapters:                 1,078,737 (0.2%)\n",
      "Reads written (passing filters):   678,686,021 (100.0%)\n",
      "\n",
      "Total basepairs processed: 50,901,451,575 bp\n",
      "Quality-trimmed:              17,099,823 bp (0.0%)\n",
      "Total written (filtered):  50,882,942,786 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 1078737 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.9%\n",
      "  C: 19.6%\n",
      "  G: 20.3%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t875027\t169671505.2\t0\t875027\n",
      "2\t136882\t42417876.3\t0\t136882\n",
      "3\t13603\t10604469.1\t0\t13603\n",
      "4\t52487\t2651117.3\t0\t52487\n",
      "5\t335\t662779.3\t0\t335\n",
      "6\t175\t165694.8\t0\t175\n",
      "7\t26\t41423.7\t0\t26\n",
      "8\t8\t10355.9\t0\t8\n",
      "9\t9\t2589.0\t0\t7 2\n",
      "10\t3\t647.2\t1\t1 2\n",
      "11\t2\t161.8\t1\t2\n",
      "12\t3\t40.5\t1\t1 2\n",
      "13\t4\t10.1\t1\t4\n",
      "14\t8\t10.1\t1\t5 3\n",
      "15\t5\t10.1\t1\t5\n",
      "16\t5\t10.1\t1\t5\n",
      "17\t6\t10.1\t1\t6\n",
      "18\t8\t10.1\t1\t5 3\n",
      "19\t5\t10.1\t1\t4 1\n",
      "20\t2\t10.1\t1\t2\n",
      "21\t4\t10.1\t1\t3 1\n",
      "22\t4\t10.1\t1\t4\n",
      "24\t4\t10.1\t1\t3 1\n",
      "25\t3\t10.1\t1\t3\n",
      "26\t7\t10.1\t1\t7\n",
      "27\t8\t10.1\t1\t7 1\n",
      "28\t6\t10.1\t1\t5 1\n",
      "29\t1\t10.1\t1\t1\n",
      "30\t2\t10.1\t1\t2\n",
      "31\t1\t10.1\t1\t1\n",
      "32\t2\t10.1\t1\t2\n",
      "33\t4\t10.1\t1\t4\n",
      "34\t2\t10.1\t1\t2\n",
      "35\t4\t10.1\t1\t3 1\n",
      "36\t3\t10.1\t1\t3\n",
      "37\t1\t10.1\t1\t1\n",
      "38\t4\t10.1\t1\t3 1\n",
      "39\t2\t10.1\t1\t2\n",
      "40\t7\t10.1\t1\t5 2\n",
      "42\t1\t10.1\t1\t0 1\n",
      "43\t12\t10.1\t1\t11 1\n",
      "44\t5\t10.1\t1\t4 1\n",
      "45\t2\t10.1\t1\t1 1\n",
      "46\t4\t10.1\t1\t3 1\n",
      "47\t5\t10.1\t1\t5\n",
      "48\t1\t10.1\t1\t1\n",
      "49\t4\t10.1\t1\t4\n",
      "50\t2\t10.1\t1\t2\n",
      "51\t1\t10.1\t1\t1\n",
      "52\t2\t10.1\t1\t1 1\n",
      "54\t3\t10.1\t1\t3\n",
      "55\t1\t10.1\t1\t1\n",
      "57\t1\t10.1\t1\t1\n",
      "58\t3\t10.1\t1\t3\n",
      "59\t1\t10.1\t1\t1\n",
      "61\t3\t10.1\t1\t3\n",
      "64\t1\t10.1\t1\t0 1\n",
      "65\t1\t10.1\t1\t1\n",
      "70\t1\t10.1\t1\t1\n",
      "71\t1\t10.1\t1\t1\n",
      "72\t1\t10.1\t1\t1\n",
      "73\t1\t10.1\t1\t1\n",
      "74\t1\t10.1\t1\t0 1\n",
      "75\t7\t10.1\t1\t4 3\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063943-4880-Lib-K-MOI-0-2-20240613/SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1.fastq.gz\n",
      "=============================================\n",
      "678686021 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t365 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 672.580 s (1.103 s/read; 54.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             609,944,511\n",
      "Reads with adapters:                 1,010,635 (0.2%)\n",
      "Reads written (passing filters):   609,944,511 (100.0%)\n",
      "\n",
      "Total basepairs processed: 45,745,838,325 bp\n",
      "Quality-trimmed:              16,444,288 bp (0.0%)\n",
      "Total written (filtered):  45,728,096,559 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 1010635 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.7%\n",
      "  C: 18.5%\n",
      "  G: 20.7%\n",
      "  T: 33.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t830298\t152486127.8\t0\t830298\n",
      "2\t125281\t38121531.9\t0\t125281\n",
      "3\t11329\t9530383.0\t0\t11329\n",
      "4\t43047\t2382595.7\t0\t43047\n",
      "5\t287\t595648.9\t0\t287\n",
      "6\t136\t148912.2\t0\t136\n",
      "7\t20\t37228.1\t0\t20\n",
      "8\t7\t9307.0\t0\t7\n",
      "9\t10\t2326.8\t0\t4 6\n",
      "10\t3\t581.7\t1\t2 1\n",
      "11\t7\t145.4\t1\t5 2\n",
      "12\t5\t36.4\t1\t4 1\n",
      "13\t4\t9.1\t1\t3 1\n",
      "14\t6\t9.1\t1\t6\n",
      "15\t4\t9.1\t1\t3 1\n",
      "16\t2\t9.1\t1\t2\n",
      "17\t8\t9.1\t1\t6 2\n",
      "18\t7\t9.1\t1\t6 1\n",
      "19\t7\t9.1\t1\t5 2\n",
      "20\t2\t9.1\t1\t2\n",
      "21\t4\t9.1\t1\t2 2\n",
      "22\t6\t9.1\t1\t6\n",
      "23\t1\t9.1\t1\t1\n",
      "24\t4\t9.1\t1\t4\n",
      "25\t5\t9.1\t1\t5\n",
      "26\t4\t9.1\t1\t4\n",
      "27\t6\t9.1\t1\t5 1\n",
      "28\t8\t9.1\t1\t6 2\n",
      "29\t3\t9.1\t1\t2 1\n",
      "30\t2\t9.1\t1\t2\n",
      "31\t3\t9.1\t1\t2 1\n",
      "32\t3\t9.1\t1\t3\n",
      "33\t3\t9.1\t1\t3\n",
      "34\t2\t9.1\t1\t1 1\n",
      "35\t3\t9.1\t1\t3\n",
      "36\t2\t9.1\t1\t2\n",
      "38\t2\t9.1\t1\t2\n",
      "39\t4\t9.1\t1\t4\n",
      "40\t10\t9.1\t1\t10\n",
      "41\t4\t9.1\t1\t4\n",
      "42\t4\t9.1\t1\t4\n",
      "43\t12\t9.1\t1\t12\n",
      "44\t4\t9.1\t1\t4\n",
      "45\t2\t9.1\t1\t2\n",
      "46\t2\t9.1\t1\t2\n",
      "47\t1\t9.1\t1\t1\n",
      "48\t4\t9.1\t1\t3 1\n",
      "49\t5\t9.1\t1\t5\n",
      "50\t10\t9.1\t1\t9 1\n",
      "51\t2\t9.1\t1\t2\n",
      "52\t3\t9.1\t1\t3\n",
      "53\t1\t9.1\t1\t1\n",
      "54\t3\t9.1\t1\t3\n",
      "55\t5\t9.1\t1\t5\n",
      "57\t3\t9.1\t1\t3\n",
      "58\t1\t9.1\t1\t1\n",
      "60\t1\t9.1\t1\t1\n",
      "61\t3\t9.1\t1\t3\n",
      "62\t1\t9.1\t1\t1\n",
      "64\t1\t9.1\t1\t1\n",
      "65\t2\t9.1\t1\t2\n",
      "66\t1\t9.1\t1\t0 1\n",
      "67\t1\t9.1\t1\t1\n",
      "69\t1\t9.1\t1\t0 1\n",
      "71\t3\t9.1\t1\t3\n",
      "72\t4\t9.1\t1\t4\n",
      "73\t2\t9.1\t1\t2\n",
      "74\t1\t9.1\t1\t1\n",
      "75\t3\t9.1\t1\t2 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063944-4882-Lib-L-MOI-0-07-20240617/SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "609944511 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t320 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t2\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 2). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 13.521 s (1.178 s/read; 50.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              11,473,790\n",
      "Reads with adapters:                    15,828 (0.1%)\n",
      "Reads written (passing filters):    11,473,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:   860,534,250 bp\n",
      "Quality-trimmed:                 342,689 bp (0.0%)\n",
      "Total written (filtered):    860,172,042 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15828 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.4%\n",
      "  C: 21.4%\n",
      "  G: 20.7%\n",
      "  T: 32.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t13325\t2868447.5\t0\t13325\n",
      "2\t1915\t717111.9\t0\t1915\n",
      "3\t133\t179278.0\t0\t133\n",
      "4\t439\t44819.5\t0\t439\n",
      "5\t7\t11204.9\t0\t7\n",
      "6\t3\t2801.2\t0\t3\n",
      "7\t2\t700.3\t0\t2\n",
      "16\t1\t0.2\t1\t0 1\n",
      "22\t1\t0.2\t1\t1\n",
      "50\t1\t0.2\t1\t1\n",
      "54\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063945-4884-Lib-K-9945-1-20240621/SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "11473790 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Illumina: 0, count Nextera: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.486 s (1.142 s/read; 52.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,183,795\n",
      "Reads with adapters:                    11,563 (0.1%)\n",
      "Reads written (passing filters):     9,183,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:   688,784,625 bp\n",
      "Quality-trimmed:                 250,994 bp (0.0%)\n",
      "Total written (filtered):    688,519,317 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11563 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.6%\n",
      "  C: 19.8%\n",
      "  G: 23.3%\n",
      "  T: 30.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t9664\t2295948.8\t0\t9664\n",
      "2\t1516\t573987.2\t0\t1516\n",
      "3\t72\t143496.8\t0\t72\n",
      "4\t301\t35874.2\t0\t301\n",
      "5\t5\t8968.5\t0\t5\n",
      "9\t1\t35.0\t0\t1\n",
      "29\t1\t0.1\t1\t1\n",
      "43\t1\t0.1\t1\t1\n",
      "45\t1\t0.1\t1\t1\n",
      "47\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063946-4885-Lib-K-9945-2-20240621/SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9183795 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.541 s (1.150 s/read; 52.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,167,786\n",
      "Reads with adapters:                    13,333 (0.1%)\n",
      "Reads written (passing filters):     9,167,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:   687,583,950 bp\n",
      "Quality-trimmed:                 296,643 bp (0.0%)\n",
      "Total written (filtered):    687,270,564 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13333 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.6%\n",
      "  C: 19.9%\n",
      "  G: 21.9%\n",
      "  T: 31.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11069\t2291946.5\t0\t11069\n",
      "2\t1748\t572986.6\t0\t1748\n",
      "3\t132\t143246.7\t0\t132\n",
      "4\t376\t35811.7\t0\t376\n",
      "5\t1\t8952.9\t0\t1\n",
      "6\t1\t2238.2\t0\t1\n",
      "36\t1\t0.1\t1\t1\n",
      "39\t1\t0.1\t1\t1\n",
      "41\t1\t0.1\t1\t1\n",
      "49\t1\t0.1\t1\t1\n",
      "50\t1\t0.1\t1\t1\n",
      "52\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063947-4886-Lib-K-9945-3-20240621/SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9167786 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Illumina: 0, count Nextera: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.625 s (1.120 s/read; 53.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,592,126\n",
      "Reads with adapters:                    13,591 (0.2%)\n",
      "Reads written (passing filters):     8,592,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:   644,409,450 bp\n",
      "Quality-trimmed:                 384,992 bp (0.1%)\n",
      "Total written (filtered):    644,007,508 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13591 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.9%\n",
      "  C: 22.1%\n",
      "  G: 20.2%\n",
      "  T: 31.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11418\t2148031.5\t0\t11418\n",
      "2\t1620\t537007.9\t0\t1620\n",
      "3\t146\t134252.0\t0\t146\n",
      "4\t395\t33563.0\t0\t395\n",
      "5\t4\t8390.7\t0\t4\n",
      "6\t1\t2097.7\t0\t1\n",
      "11\t1\t2.0\t1\t1\n",
      "22\t1\t0.1\t1\t1\n",
      "40\t2\t0.1\t1\t2\n",
      "41\t1\t0.1\t1\t1\n",
      "44\t1\t0.1\t1\t1\n",
      "50\t1\t0.1\t1\t0 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063948-4887-Lib-K-9945-4-20240617/SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8592126 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was Nextera (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.511 s (1.157 s/read; 51.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,081,546\n",
      "Reads with adapters:                    12,217 (0.1%)\n",
      "Reads written (passing filters):     9,081,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:   681,115,950 bp\n",
      "Quality-trimmed:                 265,255 bp (0.0%)\n",
      "Total written (filtered):    680,835,447 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12217 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.1%\n",
      "  C: 20.4%\n",
      "  G: 22.8%\n",
      "  T: 30.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t10218\t2270386.5\t0\t10218\n",
      "2\t1584\t567596.6\t0\t1584\n",
      "3\t77\t141899.2\t0\t77\n",
      "4\t324\t35474.8\t0\t324\n",
      "5\t6\t8868.7\t0\t6\n",
      "6\t1\t2217.2\t0\t1\n",
      "23\t1\t0.1\t1\t0 1\n",
      "24\t1\t0.1\t1\t1\n",
      "39\t1\t0.1\t1\t1\n",
      "40\t1\t0.1\t1\t1\n",
      "51\t2\t0.1\t1\t2\n",
      "71\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063949-4888-Lib-K-9945-5-20240617/SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9081546 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Illumina: 0, count Nextera: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.767 s (1.146 s/read; 52.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,522,988\n",
      "Reads with adapters:                    12,850 (0.2%)\n",
      "Reads written (passing filters):     8,522,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:   639,224,100 bp\n",
      "Quality-trimmed:                 278,707 bp (0.0%)\n",
      "Total written (filtered):    638,929,592 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 12850 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.3%\n",
      "  C: 21.9%\n",
      "  G: 20.4%\n",
      "  T: 31.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t10899\t2130747.0\t0\t10899\n",
      "2\t1494\t532686.8\t0\t1494\n",
      "3\t100\t133171.7\t0\t100\n",
      "4\t349\t33292.9\t0\t349\n",
      "6\t2\t2080.8\t0\t2\n",
      "7\t1\t520.2\t0\t1\n",
      "18\t1\t0.1\t1\t1\n",
      "32\t1\t0.1\t1\t0 1\n",
      "43\t1\t0.1\t1\t1\n",
      "45\t1\t0.1\t1\t1\n",
      "61\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063950-4889-Lib-K-9945-6-20240617/SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8522988 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count smallRNA: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 12.023 s (1.133 s/read; 52.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              10,607,688\n",
      "Reads with adapters:                    16,538 (0.2%)\n",
      "Reads written (passing filters):    10,607,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:   795,576,600 bp\n",
      "Quality-trimmed:                 334,781 bp (0.0%)\n",
      "Total written (filtered):    795,220,829 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16538 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.3%\n",
      "  C: 21.1%\n",
      "  G: 20.7%\n",
      "  T: 33.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t13816\t2651922.0\t0\t13816\n",
      "2\t1862\t662980.5\t0\t1862\n",
      "3\t198\t165745.1\t0\t198\n",
      "4\t648\t41436.3\t0\t648\n",
      "5\t5\t10359.1\t0\t5\n",
      "7\t2\t647.4\t0\t2\n",
      "8\t1\t161.9\t0\t1\n",
      "11\t1\t2.5\t1\t1\n",
      "21\t1\t0.2\t1\t1\n",
      "40\t3\t0.2\t1\t2 1\n",
      "65\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063951-4890-Lib-K-7977-7-20240621/SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "10607688 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was Nextera (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 11.868 s (1.144 s/read; 52.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              10,377,627\n",
      "Reads with adapters:                    14,894 (0.1%)\n",
      "Reads written (passing filters):    10,377,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:   778,322,025 bp\n",
      "Quality-trimmed:                 319,326 bp (0.0%)\n",
      "Total written (filtered):    777,984,025 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14894 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.1%\n",
      "  C: 20.2%\n",
      "  G: 21.9%\n",
      "  T: 30.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12429\t2594406.8\t0\t12429\n",
      "2\t1863\t648601.7\t0\t1863\n",
      "3\t142\t162150.4\t0\t142\n",
      "4\t446\t40537.6\t0\t446\n",
      "5\t4\t10134.4\t0\t4\n",
      "6\t2\t2533.6\t0\t2\n",
      "7\t2\t633.4\t0\t2\n",
      "28\t1\t0.2\t1\t1\n",
      "30\t1\t0.2\t1\t1\n",
      "40\t1\t0.2\t1\t1\n",
      "45\t1\t0.2\t1\t0 1\n",
      "49\t1\t0.2\t1\t1\n",
      "71\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063952-4891-Lib-K-7977-8-20240621/SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "10377627 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 11.046 s (1.107 s/read; 54.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,976,364\n",
      "Reads with adapters:                    15,655 (0.2%)\n",
      "Reads written (passing filters):     9,976,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:   748,227,300 bp\n",
      "Quality-trimmed:                 297,950 bp (0.0%)\n",
      "Total written (filtered):    747,909,892 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15655 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 21.4%\n",
      "  G: 19.7%\n",
      "  T: 30.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t13185\t2494091.0\t0\t13185\n",
      "2\t1770\t623522.8\t0\t1770\n",
      "3\t171\t155880.7\t0\t171\n",
      "4\t517\t38970.2\t0\t517\n",
      "5\t6\t9742.5\t0\t6\n",
      "6\t1\t2435.6\t0\t1\n",
      "7\t1\t608.9\t0\t1\n",
      "9\t1\t38.1\t0\t1\n",
      "16\t1\t0.1\t1\t1\n",
      "33\t1\t0.1\t1\t0 1\n",
      "51\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063953-4892-Lib-K-7977-9-20240621/SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9976364 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 12.447 s (1.123 s/read; 53.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              11,079,807\n",
      "Reads with adapters:                    17,396 (0.2%)\n",
      "Reads written (passing filters):    11,079,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:   830,985,525 bp\n",
      "Quality-trimmed:                 328,477 bp (0.0%)\n",
      "Total written (filtered):    830,635,191 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17396 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 21.0%\n",
      "  G: 19.5%\n",
      "  T: 31.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t14654\t2769951.8\t0\t14654\n",
      "2\t1913\t692487.9\t0\t1913\n",
      "3\t232\t173122.0\t0\t232\n",
      "4\t578\t43280.5\t0\t578\n",
      "5\t5\t10820.1\t0\t5\n",
      "6\t3\t2705.0\t0\t3\n",
      "7\t2\t676.3\t0\t2\n",
      "11\t1\t2.6\t1\t1\n",
      "20\t1\t0.2\t1\t1\n",
      "30\t1\t0.2\t1\t1\n",
      "32\t1\t0.2\t1\t1\n",
      "40\t1\t0.2\t1\t1\n",
      "43\t2\t0.2\t1\t2\n",
      "44\t1\t0.2\t1\t1\n",
      "49\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063954-4893-Lib-K-7977-10-20240617/SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "11079807 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t12\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t1\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 12). Second best hit was Nextera (count: 1)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 6.517 s (1.229 s/read; 48.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               5,303,200\n",
      "Reads with adapters:                    43,478 (0.8%)\n",
      "Reads written (passing filters):     5,303,200 (100.0%)\n",
      "\n",
      "Total basepairs processed:   397,740,000 bp\n",
      "Quality-trimmed:                 192,275 bp (0.0%)\n",
      "Total written (filtered):    397,488,999 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 43478 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.1%\n",
      "  C: 27.6%\n",
      "  G: 19.1%\n",
      "  T: 22.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t33292\t1325800.0\t0\t33292\n",
      "2\t7410\t331450.0\t0\t7410\n",
      "3\t2158\t82862.5\t0\t2158\n",
      "4\t514\t20715.6\t0\t514\n",
      "5\t17\t5178.9\t0\t17\n",
      "6\t6\t1294.7\t0\t6\n",
      "7\t7\t323.7\t0\t7\n",
      "8\t2\t80.9\t0\t2\n",
      "9\t3\t20.2\t0\t3\n",
      "10\t1\t5.1\t1\t1\n",
      "11\t2\t1.3\t1\t2\n",
      "12\t4\t0.3\t1\t4\n",
      "13\t3\t0.1\t1\t2 1\n",
      "14\t2\t0.1\t1\t1 1\n",
      "15\t4\t0.1\t1\t4\n",
      "16\t3\t0.1\t1\t2 1\n",
      "17\t4\t0.1\t1\t2 2\n",
      "19\t4\t0.1\t1\t3 1\n",
      "20\t3\t0.1\t1\t3\n",
      "21\t3\t0.1\t1\t1 2\n",
      "22\t4\t0.1\t1\t4\n",
      "23\t2\t0.1\t1\t1 1\n",
      "24\t2\t0.1\t1\t1 1\n",
      "25\t1\t0.1\t1\t1\n",
      "27\t2\t0.1\t1\t1 1\n",
      "28\t1\t0.1\t1\t1\n",
      "29\t3\t0.1\t1\t2 1\n",
      "30\t3\t0.1\t1\t3\n",
      "36\t1\t0.1\t1\t1\n",
      "37\t1\t0.1\t1\t1\n",
      "38\t1\t0.1\t1\t0 1\n",
      "39\t1\t0.1\t1\t1\n",
      "40\t1\t0.1\t1\t1\n",
      "42\t1\t0.1\t1\t1\n",
      "43\t2\t0.1\t1\t2\n",
      "44\t1\t0.1\t1\t1\n",
      "47\t3\t0.1\t1\t3\n",
      "48\t1\t0.1\t1\t1\n",
      "52\t1\t0.1\t1\t1\n",
      "60\t1\t0.1\t1\t1\n",
      "70\t1\t0.1\t1\t0 1\n",
      "75\t2\t0.1\t1\t1 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063955-4894-Lib-K-7977-11-20240617/SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "5303200 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t11 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Illumina: 0, count smallRNA: 0, count Nextera: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 11.776 s (1.129 s/read; 53.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              10,428,177\n",
      "Reads with adapters:                    13,848 (0.1%)\n",
      "Reads written (passing filters):    10,428,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:   782,113,275 bp\n",
      "Quality-trimmed:                 265,502 bp (0.0%)\n",
      "Total written (filtered):    781,830,779 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13848 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 19.5%\n",
      "  G: 20.1%\n",
      "  T: 31.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11760\t2607044.2\t0\t11760\n",
      "2\t1545\t651761.1\t0\t1545\n",
      "3\t129\t162940.3\t0\t129\n",
      "4\t404\t40735.1\t0\t404\n",
      "5\t4\t10183.8\t0\t4\n",
      "6\t2\t2545.9\t0\t2\n",
      "19\t1\t0.2\t1\t1\n",
      "21\t1\t0.2\t1\t1\n",
      "33\t1\t0.2\t1\t1\n",
      "36\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063956-4895-Lib-K-7977-12-20240617/SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "10428177 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.675 s (1.110 s/read; 54.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,818,695\n",
      "Reads with adapters:                    13,123 (0.2%)\n",
      "Reads written (passing filters):     7,818,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:   586,402,125 bp\n",
      "Quality-trimmed:                 231,734 bp (0.0%)\n",
      "Total written (filtered):    586,154,054 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 13123 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 19.5%\n",
      "  G: 19.8%\n",
      "  T: 31.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11065\t1954673.8\t0\t11065\n",
      "2\t1482\t488668.4\t0\t1482\n",
      "3\t140\t122167.1\t0\t140\n",
      "4\t423\t30541.8\t0\t423\n",
      "5\t8\t7635.4\t0\t8\n",
      "6\t1\t1908.9\t0\t1\n",
      "12\t1\t0.5\t1\t1\n",
      "37\t1\t0.1\t1\t1\n",
      "43\t1\t0.1\t1\t1\n",
      "58\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063957-4896-Lib-K-DMSO-13-20240617/SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7818695 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 12.543 s (1.175 s/read; 51.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              10,677,432\n",
      "Reads with adapters:                    18,186 (0.2%)\n",
      "Reads written (passing filters):    10,677,432 (100.0%)\n",
      "\n",
      "Total basepairs processed:   800,807,400 bp\n",
      "Quality-trimmed:                 398,263 bp (0.0%)\n",
      "Total written (filtered):    800,386,351 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 18186 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.8%\n",
      "  C: 21.4%\n",
      "  G: 19.0%\n",
      "  T: 32.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15431\t2669358.0\t0\t15431\n",
      "2\t1809\t667339.5\t0\t1809\n",
      "3\t199\t166834.9\t0\t199\n",
      "4\t732\t41708.7\t0\t732\n",
      "5\t6\t10427.2\t0\t6\n",
      "6\t6\t2606.8\t0\t6\n",
      "30\t1\t0.2\t1\t1\n",
      "43\t1\t0.2\t1\t1\n",
      "73\t1\t0.2\t1\t0 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063958-4897-Lib-K-DMSO-14-20240617/SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "10677432 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 11.880 s (1.140 s/read; 52.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:              10,418,981\n",
      "Reads with adapters:                    14,606 (0.1%)\n",
      "Reads written (passing filters):    10,418,981 (100.0%)\n",
      "\n",
      "Total basepairs processed:   781,423,575 bp\n",
      "Quality-trimmed:                 277,990 bp (0.0%)\n",
      "Total written (filtered):    781,127,574 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14606 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 21.4%\n",
      "  G: 20.3%\n",
      "  T: 29.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12424\t2604745.2\t0\t12424\n",
      "2\t1580\t651186.3\t0\t1580\n",
      "3\t130\t162796.6\t0\t130\n",
      "4\t466\t40699.1\t0\t466\n",
      "6\t2\t2543.7\t0\t2\n",
      "7\t1\t635.9\t0\t1\n",
      "40\t2\t0.2\t1\t2\n",
      "74\t1\t0.2\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063959-4898-Lib-K-DMSO-15-20240617/SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1.fastq.gz\n",
      "=============================================\n",
      "10418981 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.996 s (1.124 s/read; 53.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,006,325\n",
      "Reads with adapters:                    15,141 (0.2%)\n",
      "Reads written (passing filters):     8,006,325 (100.0%)\n",
      "\n",
      "Total basepairs processed:   600,474,375 bp\n",
      "Quality-trimmed:                 192,071 bp (0.0%)\n",
      "Total written (filtered):    600,262,925 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15141 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.5%\n",
      "  C: 15.1%\n",
      "  G: 21.8%\n",
      "  T: 30.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12411\t2001581.2\t0\t12411\n",
      "2\t1923\t500395.3\t0\t1923\n",
      "3\t169\t125098.8\t0\t169\n",
      "4\t627\t31274.7\t0\t627\n",
      "5\t6\t7818.7\t0\t6\n",
      "6\t2\t1954.7\t0\t2\n",
      "7\t1\t488.7\t0\t1\n",
      "13\t1\t0.1\t1\t1\n",
      "45\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063960-0790-Lib-LS180-9945-1/SQ24063960-0790-Lib-LS180-9945-1_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8006325 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t2 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063960-0790-Lib-LS180-9945-1_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Illumina: 0, count Nextera: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.570 s (1.117 s/read; 53.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,463,479\n",
      "Reads with adapters:                    18,978 (0.2%)\n",
      "Reads written (passing filters):     9,463,479 (100.0%)\n",
      "\n",
      "Total basepairs processed:   709,760,925 bp\n",
      "Quality-trimmed:                 221,262 bp (0.0%)\n",
      "Total written (filtered):    709,514,314 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 18978 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 16.1%\n",
      "  G: 21.3%\n",
      "  T: 33.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15260\t2365869.8\t0\t15260\n",
      "2\t2355\t591467.4\t0\t2355\n",
      "3\t270\t147866.9\t0\t270\n",
      "4\t1078\t36966.7\t0\t1078\n",
      "5\t7\t9241.7\t0\t7\n",
      "6\t2\t2310.4\t0\t2\n",
      "7\t1\t577.6\t0\t1\n",
      "17\t1\t0.1\t1\t1\n",
      "39\t1\t0.1\t1\t1\n",
      "40\t1\t0.1\t1\t1\n",
      "45\t1\t0.1\t1\t1\n",
      "62\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063961-0791-Lib-LS180-9945-2/SQ24063961-0791-Lib-LS180-9945-2_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9463479 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063961-0791-Lib-LS180-9945-2_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Illumina: 0, count Nextera: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.658 s (1.120 s/read; 53.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,513,733\n",
      "Reads with adapters:                    18,235 (0.2%)\n",
      "Reads written (passing filters):     9,513,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:   713,529,975 bp\n",
      "Quality-trimmed:                 216,256 bp (0.0%)\n",
      "Total written (filtered):    713,289,954 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 18235 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.2%\n",
      "  C: 15.8%\n",
      "  G: 21.9%\n",
      "  T: 30.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t14920\t2378433.2\t0\t14920\n",
      "2\t2301\t594608.3\t0\t2301\n",
      "3\t235\t148652.1\t0\t235\n",
      "4\t755\t37163.0\t0\t755\n",
      "5\t5\t9290.8\t0\t5\n",
      "6\t1\t2322.7\t0\t1\n",
      "7\t2\t580.7\t0\t2\n",
      "17\t1\t0.1\t1\t0 1\n",
      "19\t9\t0.1\t1\t9\n",
      "20\t1\t0.1\t1\t1\n",
      "36\t1\t0.1\t1\t1\n",
      "40\t1\t0.1\t1\t1\n",
      "43\t1\t0.1\t1\t1\n",
      "73\t2\t0.1\t1\t2\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063962-0792-Lib-LS180-9945-3/SQ24063962-0792-Lib-LS180-9945-3_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9513733 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063962-0792-Lib-LS180-9945-3_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was smallRNA (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.115 s (1.111 s/read; 53.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,200,911\n",
      "Reads with adapters:                    15,835 (0.2%)\n",
      "Reads written (passing filters):     8,200,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:   615,068,325 bp\n",
      "Quality-trimmed:                 197,107 bp (0.0%)\n",
      "Total written (filtered):    614,850,881 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15835 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 16.7%\n",
      "  G: 21.6%\n",
      "  T: 30.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12997\t2050227.8\t0\t12997\n",
      "2\t2039\t512556.9\t0\t2039\n",
      "3\t176\t128139.2\t0\t176\n",
      "4\t606\t32034.8\t0\t606\n",
      "5\t3\t8008.7\t0\t3\n",
      "6\t3\t2002.2\t0\t3\n",
      "7\t1\t500.5\t0\t1\n",
      "8\t2\t125.1\t0\t2\n",
      "9\t1\t31.3\t0\t0 1\n",
      "23\t1\t0.1\t1\t1\n",
      "25\t1\t0.1\t1\t1\n",
      "26\t1\t0.1\t1\t0 1\n",
      "30\t1\t0.1\t1\t1\n",
      "43\t2\t0.1\t1\t0 2\n",
      "55\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063963-0793-Lib-LS180-9945-4/SQ24063963-0793-Lib-LS180-9945-4_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8200911 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063963-0793-Lib-LS180-9945-4_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t3\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t3\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 3, count Illumina: 3, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.379 s (1.099 s/read; 54.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,622,402\n",
      "Reads with adapters:                    15,270 (0.2%)\n",
      "Reads written (passing filters):     7,622,402 (100.0%)\n",
      "\n",
      "Total basepairs processed:   571,680,150 bp\n",
      "Quality-trimmed:                 186,371 bp (0.0%)\n",
      "Total written (filtered):    571,473,982 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15270 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.4%\n",
      "  C: 16.2%\n",
      "  G: 21.4%\n",
      "  T: 31.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12463\t1905600.5\t0\t12463\n",
      "2\t2001\t476400.1\t0\t2001\n",
      "3\t165\t119100.0\t0\t165\n",
      "4\t623\t29775.0\t0\t623\n",
      "5\t4\t7443.8\t0\t4\n",
      "6\t6\t1860.9\t0\t6\n",
      "20\t1\t0.1\t1\t1\n",
      "25\t1\t0.1\t1\t0 1\n",
      "27\t2\t0.1\t1\t2\n",
      "33\t1\t0.1\t1\t1\n",
      "36\t1\t0.1\t1\t1\n",
      "46\t1\t0.1\t1\t1\n",
      "75\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063964-0794-Lib-LS180-9945-5/SQ24063964-0794-Lib-LS180-9945-5_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7622402 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063964-0794-Lib-LS180-9945-5_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count smallRNA: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.003 s (1.117 s/read; 53.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,062,490\n",
      "Reads with adapters:                    15,605 (0.2%)\n",
      "Reads written (passing filters):     8,062,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:   604,686,750 bp\n",
      "Quality-trimmed:                 190,875 bp (0.0%)\n",
      "Total written (filtered):    604,475,751 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 15605 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 15.9%\n",
      "  G: 21.7%\n",
      "  T: 30.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12774\t2015622.5\t0\t12774\n",
      "2\t1978\t503905.6\t0\t1978\n",
      "3\t169\t125976.4\t0\t169\n",
      "4\t670\t31494.1\t0\t670\n",
      "5\t5\t7873.5\t0\t5\n",
      "6\t3\t1968.4\t0\t3\n",
      "9\t1\t30.8\t0\t1\n",
      "12\t1\t0.5\t1\t0 1\n",
      "19\t1\t0.1\t1\t1\n",
      "23\t1\t0.1\t1\t1\n",
      "49\t1\t0.1\t1\t1\n",
      "52\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063965-0795-Lib-LS180-9945-6/SQ24063965-0795-Lib-LS180-9945-6_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8062490 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063965-0795-Lib-LS180-9945-6_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 10.182 s (1.118 s/read; 53.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               9,111,457\n",
      "Reads with adapters:                    17,387 (0.2%)\n",
      "Reads written (passing filters):     9,111,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:   683,359,275 bp\n",
      "Quality-trimmed:                 213,674 bp (0.0%)\n",
      "Total written (filtered):    683,123,342 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17387 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 16.1%\n",
      "  G: 21.6%\n",
      "  T: 30.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t14313\t2277864.2\t0\t14313\n",
      "2\t2166\t569466.1\t0\t2166\n",
      "3\t187\t142366.5\t0\t187\n",
      "4\t708\t35591.6\t0\t708\n",
      "5\t4\t8897.9\t0\t4\n",
      "6\t3\t2224.5\t0\t3\n",
      "9\t1\t34.8\t0\t1\n",
      "13\t1\t0.1\t1\t1\n",
      "25\t1\t0.1\t1\t1\n",
      "32\t2\t0.1\t1\t1 1\n",
      "72\t1\t0.1\t1\t0 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063966-0796-Lib-LS180-7977-7/SQ24063966-0796-Lib-LS180-7977-7_combined_R1.fastq.gz\n",
      "=============================================\n",
      "9111457 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t7 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063966-0796-Lib-LS180-7977-7_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.140 s (1.094 s/read; 54.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,351,512\n",
      "Reads with adapters:                    16,286 (0.2%)\n",
      "Reads written (passing filters):     8,351,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:   626,363,400 bp\n",
      "Quality-trimmed:                 178,807 bp (0.0%)\n",
      "Total written (filtered):    626,163,263 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16286 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.2%\n",
      "  C: 14.3%\n",
      "  G: 24.1%\n",
      "  T: 32.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t12977\t2087878.0\t0\t12977\n",
      "2\t2362\t521969.5\t0\t2362\n",
      "3\t169\t130492.4\t0\t169\n",
      "4\t772\t32623.1\t0\t772\n",
      "5\t2\t8155.8\t0\t2\n",
      "6\t4\t2038.9\t0\t4\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063967-0797-Lib-LS180-7977-8/SQ24063967-0797-Lib-LS180-7977-8_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8351512 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063967-0797-Lib-LS180-7977-8_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t1\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 1). Second best hit was Nextera (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.589 s (1.097 s/read; 54.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,739,787\n",
      "Reads with adapters:                    19,038 (0.2%)\n",
      "Reads written (passing filters):     8,739,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:   655,484,025 bp\n",
      "Quality-trimmed:                 243,089 bp (0.0%)\n",
      "Total written (filtered):    655,215,867 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 19038 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 15.3%\n",
      "  G: 23.4%\n",
      "  T: 31.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15383\t2184946.8\t0\t15383\n",
      "2\t2536\t546236.7\t0\t2536\n",
      "3\t289\t136559.2\t0\t289\n",
      "4\t801\t34139.8\t0\t801\n",
      "5\t4\t8534.9\t0\t4\n",
      "6\t6\t2133.7\t0\t6\n",
      "7\t2\t533.4\t0\t2\n",
      "10\t2\t8.3\t1\t1 1\n",
      "11\t2\t2.1\t1\t2\n",
      "12\t1\t0.5\t1\t1\n",
      "20\t1\t0.1\t1\t1\n",
      "21\t2\t0.1\t1\t2\n",
      "23\t1\t0.1\t1\t1\n",
      "25\t1\t0.1\t1\t1\n",
      "30\t1\t0.1\t1\t1\n",
      "31\t1\t0.1\t1\t1\n",
      "32\t2\t0.1\t1\t2\n",
      "34\t1\t0.1\t1\t1\n",
      "75\t2\t0.1\t1\t0 2\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063968-0798-Lib-LS180-7977-9/SQ24063968-0798-Lib-LS180-7977-9_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8739787 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t8 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063968-0798-Lib-LS180-7977-9_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.625 s (1.112 s/read; 53.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,658,219\n",
      "Reads with adapters:                    18,374 (0.2%)\n",
      "Reads written (passing filters):     8,658,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:   649,366,425 bp\n",
      "Quality-trimmed:                 187,938 bp (0.0%)\n",
      "Total written (filtered):    649,154,810 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 18374 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 13.5%\n",
      "  G: 22.1%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t14930\t2164554.8\t0\t14930\n",
      "2\t2469\t541138.7\t0\t2469\n",
      "3\t167\t135284.7\t0\t167\n",
      "4\t798\t33821.2\t0\t798\n",
      "5\t6\t8455.3\t0\t6\n",
      "7\t1\t528.5\t0\t1\n",
      "8\t1\t132.1\t0\t1\n",
      "11\t1\t2.1\t1\t1\n",
      "60\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063969-0799-Lib-LS180-7977-10/SQ24063969-0799-Lib-LS180-7977-10_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8658219 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t4 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063969-0799-Lib-LS180-7977-10_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t3\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 3). Second best hit was Nextera (count: 0)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.605 s (1.118 s/read; 53.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,699,569\n",
      "Reads with adapters:                    16,325 (0.2%)\n",
      "Reads written (passing filters):     7,699,569 (100.0%)\n",
      "\n",
      "Total basepairs processed:   577,467,675 bp\n",
      "Quality-trimmed:                 166,609 bp (0.0%)\n",
      "Total written (filtered):    577,278,611 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 16325 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.2%\n",
      "  C: 15.4%\n",
      "  G: 22.3%\n",
      "  T: 32.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t13144\t1924892.2\t0\t13144\n",
      "2\t2280\t481223.1\t0\t2280\n",
      "3\t175\t120305.8\t0\t175\n",
      "4\t678\t30076.4\t0\t678\n",
      "5\t15\t7519.1\t0\t15\n",
      "6\t2\t1879.8\t0\t2\n",
      "8\t2\t117.5\t0\t2\n",
      "9\t1\t29.4\t0\t0 1\n",
      "13\t1\t0.1\t1\t1\n",
      "17\t1\t0.1\t1\t1\n",
      "33\t1\t0.1\t1\t1\n",
      "37\t2\t0.1\t1\t2\n",
      "40\t1\t0.1\t1\t1\n",
      "42\t2\t0.1\t1\t2\n",
      "45\t1\t0.1\t1\t1\n",
      "48\t1\t0.1\t1\t1\n",
      "49\t1\t0.1\t1\t1\n",
      "51\t1\t0.1\t1\t1\n",
      "52\t1\t0.1\t1\t1\n",
      "54\t2\t0.1\t1\t2\n",
      "55\t3\t0.1\t1\t3\n",
      "56\t1\t0.1\t1\t0 1\n",
      "58\t1\t0.1\t1\t1\n",
      "59\t1\t0.1\t1\t1\n",
      "61\t3\t0.1\t1\t3\n",
      "64\t3\t0.1\t1\t3\n",
      "75\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063970-0800-Lib-LS180-7977-11/SQ24063970-0800-Lib-LS180-7977-11_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7699569 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t15 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063970-0800-Lib-LS180-7977-11_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count Nextera: 0, count Illumina: 0, count smallRNA: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.134 s (1.110 s/read; 54.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,325,250\n",
      "Reads with adapters:                    14,982 (0.2%)\n",
      "Reads written (passing filters):     7,325,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:   549,393,750 bp\n",
      "Quality-trimmed:                 158,674 bp (0.0%)\n",
      "Total written (filtered):    549,214,419 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14982 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.7%\n",
      "  C: 15.1%\n",
      "  G: 23.6%\n",
      "  T: 35.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11698\t1831312.5\t0\t11698\n",
      "2\t2161\t457828.1\t0\t2161\n",
      "3\t195\t114457.0\t0\t195\n",
      "4\t916\t28614.3\t0\t916\n",
      "5\t2\t7153.6\t0\t2\n",
      "6\t1\t1788.4\t0\t1\n",
      "7\t1\t447.1\t0\t1\n",
      "12\t1\t0.4\t1\t1\n",
      "24\t1\t0.1\t1\t1\n",
      "40\t1\t0.1\t1\t1\n",
      "44\t1\t0.1\t1\t1\n",
      "54\t2\t0.1\t1\t2\n",
      "62\t1\t0.1\t1\t1\n",
      "75\t1\t0.1\t1\t0 1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063971-0801-Lib-LS180-7977-12/SQ24063971-0801-Lib-LS180-7977-12_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7325250 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063971-0801-Lib-LS180-7977-12_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t14\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Nextera\t10\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 14). Second best hit was Nextera (count: 10)\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 7.871 s (1.120 s/read; 53.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,025,896\n",
      "Reads with adapters:                    14,891 (0.2%)\n",
      "Reads written (passing filters):     7,025,896 (100.0%)\n",
      "\n",
      "Total basepairs processed:   526,942,200 bp\n",
      "Quality-trimmed:                 146,913 bp (0.0%)\n",
      "Total written (filtered):    526,775,186 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14891 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 15.5%\n",
      "  G: 24.3%\n",
      "  T: 30.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11778\t1756474.0\t0\t11778\n",
      "2\t2238\t439118.5\t0\t2238\n",
      "3\t207\t109779.6\t0\t207\n",
      "4\t627\t27444.9\t0\t627\n",
      "5\t6\t6861.2\t0\t6\n",
      "6\t2\t1715.3\t0\t2\n",
      "7\t1\t428.8\t0\t1\n",
      "10\t1\t6.7\t1\t1\n",
      "15\t1\t0.1\t1\t1\n",
      "17\t1\t0.1\t1\t1\n",
      "19\t21\t0.1\t1\t21\n",
      "20\t3\t0.1\t1\t3\n",
      "24\t1\t0.1\t1\t1\n",
      "31\t1\t0.1\t1\t1\n",
      "32\t2\t0.1\t1\t2\n",
      "49\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063972-0802-Lib-LS180-DMSO-13/SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7025896 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t3 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 8.588 s (1.179 s/read; 50.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               7,287,386\n",
      "Reads with adapters:                    14,359 (0.2%)\n",
      "Reads written (passing filters):     7,287,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:   546,553,950 bp\n",
      "Quality-trimmed:                 157,698 bp (0.0%)\n",
      "Total written (filtered):    546,377,281 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 14359 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 14.7%\n",
      "  G: 23.7%\n",
      "  T: 32.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11384\t1821846.5\t0\t11384\n",
      "2\t2148\t455461.6\t0\t2148\n",
      "3\t148\t113865.4\t0\t148\n",
      "4\t666\t28466.4\t0\t666\n",
      "5\t5\t7116.6\t0\t5\n",
      "6\t3\t1779.1\t0\t3\n",
      "7\t1\t444.8\t0\t1\n",
      "21\t1\t0.1\t1\t1\n",
      "27\t1\t0.1\t1\t1\n",
      "38\t1\t0.1\t1\t1\n",
      "47\t1\t0.1\t1\t1\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063973-0804-Lib-LS180-DMSO-14/SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1.fastq.gz\n",
      "=============================================\n",
      "7287386 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t6 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1_trimmed.fq.gz\n",
      "Running Trim Galore on 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "Cutadapt version: 4.9\n",
      "Cutadapt seems to be using Python 3! Proceeding with multi-core enabled Cutadapt using 8 cores\n",
      "Parallel gzip (pigz) detected. Proceeding with multicore (de)compression using 8 cores\n",
      "\n",
      "Proceeding with 'pigz -p 4' for decompression\n",
      "To decrease CPU usage of decompression, please install 'igzip' and run again\n",
      "\n",
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Output will be written into the directory: /opt/notebooks/02_trim_galore/\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "smallRNA\t0\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Illumina\t0\tAGATCGGAAGAGC\t1000000\t0.00\n",
      "Unable to auto-detect most prominent adapter from the first specified file (count smallRNA: 0, count Nextera: 0, count Illumina: 0)\n",
      "Defaulting to Illumina universal adapter ( AGATCGGAAGAGC ). Specify -a SEQUENCE to avoid this behavior).\n",
      "\n",
      "pigz: abort: write error on <stdout> (Broken pipe)\n",
      "Writing report to '/opt/notebooks/02_trim_galore/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Trimming mode: single-end\n",
      "Trim Galore version: 0.6.10\n",
      "Cutadapt version: 4.9\n",
      "Python version: 3.9.16\n",
      "Number of cores used for trimming: 8\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; default (inconclusive auto-detection))\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length before a sequence gets removed: 20 bp\n",
      "Running FastQC on the data once trimming has completed\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Cutadapt seems to be fairly up-to-date (version 4.9). Setting -j 8\n",
      "Writing final adapter and quality trimmed output to SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz <<< \n",
      "This is cutadapt 4.9 with Python 3.9.16\n",
      "Command line parameters: -j 8 -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "Processing single-end reads on 8 cores ...\n",
      "Finished in 9.787 s (1.105 s/read; 54.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               8,858,985\n",
      "Reads with adapters:                    17,591 (0.2%)\n",
      "Reads written (passing filters):     8,858,985 (100.0%)\n",
      "\n",
      "Total basepairs processed:   664,423,875 bp\n",
      "Quality-trimmed:                 191,820 bp (0.0%)\n",
      "Total written (filtered):    664,208,863 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 17591 times\n",
      "\n",
      "Minimum overlap: 1\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.3%\n",
      "  C: 14.7%\n",
      "  G: 24.4%\n",
      "  T: 31.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t13917\t2214746.2\t0\t13917\n",
      "2\t2698\t553686.6\t0\t2698\n",
      "3\t156\t138421.6\t0\t156\n",
      "4\t808\t34605.4\t0\t808\n",
      "5\t3\t8651.4\t0\t3\n",
      "6\t2\t2162.8\t0\t2\n",
      "7\t2\t540.7\t0\t2\n",
      "8\t2\t135.2\t0\t2\n",
      "22\t1\t0.1\t1\t1\n",
      "50\t2\t0.1\t1\t2\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: 20240827_abundant_emu/s252x10006_WZY_03020703194240322A-20240822-L-02-2024-08-261356/Sample_SQ24063974-0806-Lib-LS180-DMSO-15/SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1.fastq.gz\n",
      "=============================================\n",
      "8858985 sequences processed in total\n",
      "Sequences removed because they became shorter than the length cutoff of 20 bp:\t5 (0.0%)\n",
      "\n",
      "\n",
      "  >>> Now running FastQC on the data <<<\n",
      "\n",
      "Started analysis of SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 5% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 10% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 15% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 20% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 25% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 30% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 35% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 40% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 45% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 50% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 55% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 60% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 65% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 70% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 75% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 80% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 85% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 90% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Approx 95% complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Analysis complete for SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1_trimmed.fq.gz\n",
      "Fri Aug 30 01:34:26 UTC 2024\n"
     ]
    }
   ],
   "source": [
    "!date\n",
    "!./run_trim_galore.sh\n",
    "!date"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "feea7a90-ae90-4218-ac67-866c3f025c97",
   "metadata": {},
   "source": [
    "## multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "474d0a7e-5311-46ad-887c-63cf1ecb0735",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                                                \n",
      " \u001b[91m///\u001b[0m \u001b]8;id=465601;https://multiqc.info\u001b\\\u001b[1mMultiQC\u001b[0m\u001b]8;;\u001b\\  \u001b[2mv1.24.1\u001b[0m                                                         \n",
      "                                                                                \n",
      " \u001b[33mUsage:\u001b[0m \u001b[1mmultiqc\u001b[0m [\u001b[1;36mOPTIONS\u001b[0m] [\u001b[1;36mANALYSIS\u001b[0m \u001b[1;36mDIRECTORY\u001b[0m]                                  \n",
      "                                                                                \n",
      " MultiQC aggregates results from bioinformatics analyses across many samples    \n",
      " into a single report.                                                          \n",
      " \u001b[2mIt searches a given directory for analysis logs and compiles an HTML report. \u001b[0m  \n",
      " \u001b[2mIt's a general use tool, perfect for summarising the output from numerous \u001b[0m     \n",
      " \u001b[2mbioinformatics tools.\u001b[0m                                                          \n",
      " \u001b[2mTo run, supply with one or more directory to scan for analysis results. For \u001b[0m   \n",
      " \u001b[2mexample, to run in the current working directory, use '\u001b[0m\u001b[1;2;34mmultiqc .\u001b[0m\u001b[2m'\u001b[0m              \n",
      "                                                                                \n",
      "\u001b[2m\u001b[0m\u001b[2m Main options \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--force\u001b[0m            \u001b[1;32m-f\u001b[0m  Overwrite any existing reports                        \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--config\u001b[0m           \u001b[1;32m-c\u001b[0m  Specific config file to load, after those in MultiQC  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        dir / home dir / working dir.                         \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(PATH)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--cl-config\u001b[0m            Specify MultiQC config YAML on the command line       \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(TEXT)                                         \u001b[0m       \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--filename\u001b[0m         \u001b[1;32m-n\u001b[0m  Report filename. Use '\u001b[3;33mstdout\u001b[0m' to print to standard    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        out.                                                  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(TEXT)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--outdir\u001b[0m           \u001b[1;32m-o\u001b[0m  Create report in the specified output directory.      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(TEXT)                                          \u001b[0m      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--ignore\u001b[0m           \u001b[1;32m-x\u001b[0m  Ignore analysis files \u001b[2;33m(GLOB EXPRESSION)\u001b[0m               \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--ignore-samples\u001b[0m       Ignore sample names \u001b[2;33m(GLOB EXPRESSION)\u001b[0m                 \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--ignore-symlinks\u001b[0m      Ignore symlinked directories and files                \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--file-list\u001b[0m        \u001b[1;32m-l\u001b[0m  Supply a file containing a list of file paths to be   \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        searched, one per row                                 \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m Choosing modules to run \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--module\u001b[0m   \u001b[1;32m-m\u001b[0m  Use only this module. Can specify multiple times.             \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                \u001b[2;33m(MODULE NAME)                                    \u001b[0m             \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--exclude\u001b[0m  \u001b[1;32m-e\u001b[0m  Do not use this module. Can specify multiple times.           \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                \u001b[2;33m(MODULE NAME)                                      \u001b[0m           \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m Sample handling \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--dirs\u001b[0m           \u001b[1;32m-d\u001b[0m   Prepend directory to sample names                      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--dirs-depth\u001b[0m     \u001b[1;32m-dd\u001b[0m  Prepend \u001b[3;33mn\u001b[0m directories to sample names. Negative number \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                       to take from start of path.                            \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                       \u001b[2;33m(INTEGER)                                             \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--fullnames\u001b[0m      \u001b[1;32m-s\u001b[0m   Do not clean the sample names \u001b[3m(leave as full file \u001b[0m     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                       \u001b[3mname)\u001b[0m                                                  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--fn_as_s_name\u001b[0m        Use the log filename as the sample name                \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--replace-names\u001b[0m       TSV file to rename sample names during report          \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                       generation                                             \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                       \u001b[2;33m(PATH)                                                \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m Report customisation \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--title\u001b[0m            \u001b[1;32m-i\u001b[0m  Report title. Printed as page header, used for        \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        filename if not otherwise specified.                  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(TEXT)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--comment\u001b[0m          \u001b[1;32m-b\u001b[0m  Custom comment, will be printed at the top of the     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        report.                                               \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(TEXT)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--template\u001b[0m         \u001b[1;32m-t\u001b[0m  Report template to use.                               \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(default|gathered|geo|sections|simple)\u001b[0m                \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--sample-names\u001b[0m         TSV file containing alternative sample names for      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        renaming buttons in the report                        \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(PATH)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--sample-filters\u001b[0m       TSV file containing show/hide patterns for the report \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                        \u001b[2;33m(PATH)                                               \u001b[0m \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--custom-css-file\u001b[0m      Custom CSS file to add to the final report \u001b[2;33m(PATH)\u001b[0m     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m Output files \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--flat\u001b[0m                    \u001b[1;32m-fp\u001b[0m  Use only flat plots \u001b[3m(static images)\u001b[0m           \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--interactive\u001b[0m             \u001b[1;32m-ip\u001b[0m  Use only interactive plots \u001b[3m(in-browser \u001b[0m       \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                \u001b[3mJavascript)\u001b[0m                                   \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--export\u001b[0m                  \u001b[1;32m-p\u001b[0m   Export plots as static images in addition to  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                the report                                    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--data-dir\u001b[0m/\u001b[1;36m--no-data-dir\u001b[0m       Force the parsed data directory to be         \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                created.                                      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--data-format\u001b[0m             \u001b[1;32m-k\u001b[0m   Output parsed data in a different format.     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                \u001b[2;33m(tsv|csv|json|yaml)                      \u001b[0m     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--zip-data-dir\u001b[0m            \u001b[1;32m-z\u001b[0m   Compress the data directory.                  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--no-report\u001b[0m                    Do not generate a report, only export data    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                and plots                                     \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--pdf\u001b[0m                          Creates PDF report with the \u001b[3m'simple'\u001b[0m          \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                                template. Requires \u001b]8;id=373923;https://pandoc.org/\u001b\\Pandoc\u001b]8;;\u001b\\ to be installed.    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m MultiQC behaviour \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--verbose\u001b[0m            \u001b[1;32m-v\u001b[0m  Increase output verbosity. \u001b[2;33m(INTEGER RANGE)\u001b[0m          \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--quiet\u001b[0m              \u001b[1;32m-q\u001b[0m  Only show log warnings                              \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--strict\u001b[0m                 Don't catch exceptions, run additional code checks  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          to help development.                                \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--development\u001b[0m,\u001b[1;36m--dev\u001b[0m      Development mode. Do not compress and minimise JS,  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          export uncompressed plot data                       \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--require-logs\u001b[0m           Require all explicitly requested modules to have    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          log files. If not, MultiQC will exit with an error. \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--profile-runtime\u001b[0m        Add analysis of how long MultiQC takes to run to    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          the report                                          \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--profile-memory\u001b[0m         Add analysis of how much memory each module uses.   \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          Note that tracking memory will increase the         \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          runtime, so the runtime metrics could scale up a    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          few times                                           \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--no-megaqc-upload\u001b[0m       Don't upload generated report to MegaQC, even if    \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                          MegaQC options are found                            \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--no-ansi\u001b[0m                Disable coloured log output                         \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--version\u001b[0m                Show the version and exit.                          \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--help\u001b[0m               \u001b[1;32m-h\u001b[0m  Show this message and exit.                         \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\u001b[2m Options \u001b[0m\u001b[2m\u001b[0m\u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--clean-up\u001b[0m/\u001b[1;36m--no-clean-up\u001b[0m    Remove the temporary directory and log file      \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                             after finishing                                  \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m \u001b[1;36m--no-version-check\u001b[0m          Disable checking the latest MultiQC version on   \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m                             the server                                       \u001b[2m\u001b[0m\n",
      "\u001b[2m\u001b[0m\n",
      "                                                                                \n",
      " See \u001b]8;id=398711;http://multiqc.info\u001b\\http://multiqc.info\u001b]8;;\u001b\\ for more details.                                      \n",
      "\n"
     ]
    }
   ],
   "source": [
    "!multiqc --help"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "16c9a391-e2b7-4efa-95c2-5d8a2ef06961",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "!mkdir -p 03_multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "47ba1eb4-578b-4584-973c-b02ce4ace5a8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fri Aug 30 01:44:17 UTC 2024\n",
      "\n",
      "\u001b[91m///\u001b[0m \u001b]8;id=766116;https://multiqc.info\u001b\\\u001b[1mMultiQC\u001b[0m\u001b]8;;\u001b\\  \u001b[2mv1.24.1\u001b[0m\n",
      "\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.tmp_dir                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mUsing new temporary directory: /tmp/tmpq0ps_0h9\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mroot                                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLogging to file: /tmp/tmpq0ps_0h9/multiqc.log\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mThis is MultiQC v1.24.1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning Python 3.9.16 (main, Mar  8 2023, 14:00:05)  [GCC 11.2.0]\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mStarting new HTTPS connection (1): api.multiqc.info:443\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mhttps://api.multiqc.info:443 \"GET /version?version_multiqc=1.24.1&version_python=3.9.16&operating_system=Linux&is_docker=True&is_singularity=False&is_conda=True&is_ci=False&is_notebook=False HTTP/1.1\" 200 182\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.version_check                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLatest MultiQC version is v1.24.1, released 2024-08-21\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWorking dir : /opt/notebooks\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mTemplate    : default\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCommand used: /opt/conda/bin/multiqc 01_fastqc 02_trim_galore -o 03_multiqc --verbose\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Search path: /opt/notebooks/01_fastqc\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Search path: /opt/notebooks/02_trim_galore\n",
      "\u001b[32m[2024-08-30 01:44:19]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mAnalysing modules: custom_content, ccs, ngsderive, purple, conpair, isoseq, lima, peddy, somalier, methylqa, mosdepth, phantompeakqualtools, qualimap, bamdst, preseq, hifiasm, quast, qorts, rna_seqc, rockhopper, rsem, rseqc, busco, bustools, goleft_indexcov, gffcompare, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, mirtop, glimpse, gopeaks, homer, hops, macs2, theta2, snpeff, gatk, htseq, bcftools, featurecounts, fgbio, dragen, dragen_fastqc, dedup, pbmarkdup, damageprofiler, mapdamage, biobambam2, jcvi, mtnucratio, picard, vep, bakta, prokka, qc3C, nanoq, nanostat, samblaster, samtools, bamtools, sambamba, ngsbits, pairtools, sexdeterrmine, seqera_cli, eigenstratdatabasetools, jellyfish, vcftools, longranger, stacks, varscan2, snippy, umitools, truvari, megahit, ganon, bbmap, bismark, biscuit, diamond, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, hostile, cellranger, snpsplit, odgi, vg, pangolin, nextclade, freyja, humid, kat, leehom, librarian, nonpareil, adapterremoval, bbduk, clipandmerge, cutadapt, flexbar, sourmash, kaiju, bracken, kraken, malt, motus, trimmomatic, sickle, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, sequali, filtlong, prinseqplusplus, pychopper, porechop, pycoqc, minionqc, anglerfish, multivcfanalyzer, clusterflow, checkqc, bcl2fastq, bclconvert, interop, ivar, flash, seqyclean, optitype, whatshap, spaceranger, xenome, xengsort, metaphlan, seqwho, software_versions\u001b[0m\n",
      "\u001b[2K         \u001b[34msearching\u001b[0m | \u001b[90m\u001b[0m \u001b[35m100%\u001b[0m \u001b[32m192/192\u001b[0m  [0m0m  \n",
      "\u001b[?25h\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mSummary of files that were skipped by the search: |skipped_filesize_limit: 32|, |skipped_module_specific_max_filesize: 96|, |skipped_no_match: 96|\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mNo samples found: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: cutadapt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.modules.cutadapt.cutadapt                 \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Found 32 reports\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_cutadapt.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_data\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_counts\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_exp\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_obsexp\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: fastqc\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063943-4880-Lib-K-MOI-0-2-20240613_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:21]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063944-4882-Lib-L-MOI-0-07-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063945-4884-Lib-K-9945-1-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063946-4885-Lib-K-9945-2-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063947-4886-Lib-K-9945-3-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063948-4887-Lib-K-9945-4-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063949-4888-Lib-K-9945-5-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063950-4889-Lib-K-9945-6-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063951-4890-Lib-K-7977-7-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063952-4891-Lib-K-7977-8-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063953-4892-Lib-K-7977-9-20240621_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063954-4893-Lib-K-7977-10-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063955-4894-Lib-K-7977-11-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063956-4895-Lib-K-7977-12-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063957-4896-Lib-K-DMSO-13-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063958-4897-Lib-K-DMSO-14-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063959-4898-Lib-K-DMSO-15-20240617_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063960-0790-Lib-LS180-9945-1_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063961-0791-Lib-LS180-9945-2_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063962-0792-Lib-LS180-9945-3_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063963-0793-Lib-LS180-9945-4_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063964-0794-Lib-LS180-9945-5_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063965-0795-Lib-LS180-9945-6_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063966-0796-Lib-LS180-7977-7_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063967-0797-Lib-LS180-7977-8_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063968-0798-Lib-LS180-7977-9_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063969-0799-Lib-LS180-7977-10_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063970-0800-Lib-LS180-7977-11_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063971-0801-Lib-LS180-7977-12_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063972-0802-Lib-LS180-DMSO-13_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063973-0804-Lib-LS180-DMSO-14_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mDuplicate sample name found! Overwriting: SQ24063974-0806-Lib-LS180-DMSO-15_combined_R1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Found 32 reports\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_fastqc.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.order_of_duplication_levels\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.status_colours\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.software_versions                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReading software versions from config.software_versions\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_software_versions.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRendering plots\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_general_stats.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mExporting plot data to files\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_filtered_reads_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_trimmed_sequences_plot_3_Counts.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_trimmed_sequences_plot_3_Obs_Exp.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_counts_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_sequence_quality_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_quality_scores_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Percentages.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Counts.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_n_content_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_length_distribution_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_duplication_levels_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_overrepresented_sequences_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_top_overrepresented_sequences_table.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc-status-check-heatmap.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mMoving data file from '/tmp/tmpq0ps_0h9/multiqc_data' to '/opt/notebooks/03_multiqc/multiqc_data'\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Data        : 03_multiqc/multiqc_data\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCompressing plot data\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Report      : 03_multiqc/multiqc_report.html\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReport HTML written to /opt/notebooks/03_multiqc/multiqc_report.html\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:44:22]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  MultiQC complete\n",
      "Fri Aug 30 01:44:23 UTC 2024\n"
     ]
    }
   ],
   "source": [
    "!date\n",
    "!multiqc 01_fastqc 02_trim_galore -o 03_multiqc --verbose\n",
    "!date"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "2171db4e-ae5b-4c48-ab95-7ad6bfd5597c",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fri Aug 30 01:50:59 UTC 2024\n",
      "\n",
      "\u001b[91m///\u001b[0m \u001b]8;id=690879;https://multiqc.info\u001b\\\u001b[1mMultiQC\u001b[0m\u001b]8;;\u001b\\  \u001b[2mv1.24.1\u001b[0m\n",
      "\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.tmp_dir                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mUsing new temporary directory: /tmp/tmplk9ilnkt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mroot                                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLogging to file: /tmp/tmplk9ilnkt/multiqc.log\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mThis is MultiQC v1.24.1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning Python 3.9.16 (main, Mar  8 2023, 14:00:05)  [GCC 11.2.0]\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mStarting new HTTPS connection (1): api.multiqc.info:443\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mhttps://api.multiqc.info:443 \"GET /version?version_multiqc=1.24.1&version_python=3.9.16&operating_system=Linux&is_docker=True&is_singularity=False&is_conda=True&is_ci=False&is_notebook=False HTTP/1.1\" 200 182\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.version_check                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLatest MultiQC version is v1.24.1, released 2024-08-21\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWorking dir : /opt/notebooks\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mTemplate    : default\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCommand used: /opt/conda/bin/multiqc 01_fastqc -o 03_multiqc_01_fastqc --verbose\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Search path: /opt/notebooks/01_fastqc\n",
      "\u001b[32m[2024-08-30 01:51:01]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mAnalysing modules: custom_content, ccs, ngsderive, purple, conpair, isoseq, lima, peddy, somalier, methylqa, mosdepth, phantompeakqualtools, qualimap, bamdst, preseq, hifiasm, quast, qorts, rna_seqc, rockhopper, rsem, rseqc, busco, bustools, goleft_indexcov, gffcompare, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, mirtop, glimpse, gopeaks, homer, hops, macs2, theta2, snpeff, gatk, htseq, bcftools, featurecounts, fgbio, dragen, dragen_fastqc, dedup, pbmarkdup, damageprofiler, mapdamage, biobambam2, jcvi, mtnucratio, picard, vep, bakta, prokka, qc3C, nanoq, nanostat, samblaster, samtools, bamtools, sambamba, ngsbits, pairtools, sexdeterrmine, seqera_cli, eigenstratdatabasetools, jellyfish, vcftools, longranger, stacks, varscan2, snippy, umitools, truvari, megahit, ganon, bbmap, bismark, biscuit, diamond, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, hostile, cellranger, snpsplit, odgi, vg, pangolin, nextclade, freyja, humid, kat, leehom, librarian, nonpareil, adapterremoval, bbduk, clipandmerge, cutadapt, flexbar, sourmash, kaiju, bracken, kraken, malt, motus, trimmomatic, sickle, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, sequali, filtlong, prinseqplusplus, pychopper, porechop, pycoqc, minionqc, anglerfish, multivcfanalyzer, clusterflow, checkqc, bcl2fastq, bclconvert, interop, ivar, flash, seqyclean, optitype, whatshap, spaceranger, xenome, xengsort, metaphlan, seqwho, software_versions\u001b[0m\n",
      "\u001b[2K         \u001b[34msearching\u001b[0m | \u001b[90m\u001b[0m \u001b[35m100%\u001b[0m \u001b[32m64/64\u001b[0m  l\u001b[0mm  \n",
      "\u001b[?25h\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mSummary of files that were skipped by the search: |skipped_module_specific_max_filesize: 32|, |skipped_no_match: 32|\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mNo samples found: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: fastqc\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Found 32 reports\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_fastqc.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.order_of_duplication_levels\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.status_colours\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.software_versions                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReading software versions from config.software_versions\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_software_versions.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRendering plots\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_general_stats.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mExporting plot data to files\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_counts_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_sequence_quality_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_quality_scores_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Percentages.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Counts.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_n_content_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_duplication_levels_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_overrepresented_sequences_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_top_overrepresented_sequences_table.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc-status-check-heatmap.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mMoving data file from '/tmp/tmplk9ilnkt/multiqc_data' to '/opt/notebooks/03_multiqc_01_fastqc/multiqc_data'\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Data        : 03_multiqc_01_fastqc/multiqc_data\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCompressing plot data\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Report      : 03_multiqc_01_fastqc/multiqc_report.html\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReport HTML written to /opt/notebooks/03_multiqc_01_fastqc/multiqc_report.html\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:02]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  MultiQC complete\n",
      "Fri Aug 30 01:51:03 UTC 2024\n"
     ]
    }
   ],
   "source": [
    "!date\n",
    "!multiqc 01_fastqc -o 03_multiqc_01_fastqc --verbose\n",
    "!date"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "c36ba988-de23-4d00-82c9-dfc5f946af55",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fri Aug 30 01:51:48 UTC 2024\n",
      "\n",
      "\u001b[91m///\u001b[0m \u001b]8;id=23043;https://multiqc.info\u001b\\\u001b[1mMultiQC\u001b[0m\u001b]8;;\u001b\\  \u001b[2mv1.24.1\u001b[0m\n",
      "\n",
      "\u001b[32m[2024-08-30 01:51:50]\u001b[0m \u001b[34mmultiqc.core.tmp_dir                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mUsing new temporary directory: /tmp/tmpo_xhmpnl\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:50]\u001b[0m \u001b[34mroot                                              \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLogging to file: /tmp/tmpo_xhmpnl/multiqc.log\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:50]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mThis is MultiQC v1.24.1\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:50]\u001b[0m \u001b[34mmultiqc.core.update_config                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning Python 3.9.16 (main, Mar  8 2023, 14:00:05)  [GCC 11.2.0]\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:50]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mStarting new HTTPS connection (1): api.multiqc.info:443\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34murllib3.connectionpool                            \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mhttps://api.multiqc.info:443 \"GET /version?version_multiqc=1.24.1&version_python=3.9.16&operating_system=Linux&is_docker=True&is_singularity=False&is_conda=True&is_ci=False&is_notebook=False HTTP/1.1\" 200 182\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.core.version_check                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mLatest MultiQC version is v1.24.1, released 2024-08-21\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWorking dir : /opt/notebooks\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mTemplate    : default\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCommand used: /opt/conda/bin/multiqc 02_trim_galore -o 03_multiqc_02_trim_galore --verbose\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Search path: /opt/notebooks/02_trim_galore\n",
      "\u001b[32m[2024-08-30 01:51:51]\u001b[0m \u001b[34mmultiqc.core.file_search                          \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mAnalysing modules: custom_content, ccs, ngsderive, purple, conpair, isoseq, lima, peddy, somalier, methylqa, mosdepth, phantompeakqualtools, qualimap, bamdst, preseq, hifiasm, quast, qorts, rna_seqc, rockhopper, rsem, rseqc, busco, bustools, goleft_indexcov, gffcompare, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, mirtop, glimpse, gopeaks, homer, hops, macs2, theta2, snpeff, gatk, htseq, bcftools, featurecounts, fgbio, dragen, dragen_fastqc, dedup, pbmarkdup, damageprofiler, mapdamage, biobambam2, jcvi, mtnucratio, picard, vep, bakta, prokka, qc3C, nanoq, nanostat, samblaster, samtools, bamtools, sambamba, ngsbits, pairtools, sexdeterrmine, seqera_cli, eigenstratdatabasetools, jellyfish, vcftools, longranger, stacks, varscan2, snippy, umitools, truvari, megahit, ganon, bbmap, bismark, biscuit, diamond, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, hostile, cellranger, snpsplit, odgi, vg, pangolin, nextclade, freyja, humid, kat, leehom, librarian, nonpareil, adapterremoval, bbduk, clipandmerge, cutadapt, flexbar, sourmash, kaiju, bracken, kraken, malt, motus, trimmomatic, sickle, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, sequali, filtlong, prinseqplusplus, pychopper, porechop, pycoqc, minionqc, anglerfish, multivcfanalyzer, clusterflow, checkqc, bcl2fastq, bclconvert, interop, ivar, flash, seqyclean, optitype, whatshap, spaceranger, xenome, xengsort, metaphlan, seqwho, software_versions\u001b[0m\n",
      "\u001b[2K         \u001b[34msearching\u001b[0m | \u001b[90m\u001b[0m \u001b[35m100%\u001b[0m \u001b[32m128/128\u001b[0m  [0m0m  \n",
      "\u001b[?25h\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mSummary of files that were skipped by the search: |skipped_filesize_limit: 32|, |skipped_module_specific_max_filesize: 64|, |skipped_no_match: 64|\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mNo samples found: custom_content\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: cutadapt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.modules.cutadapt.cutadapt                 \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Found 32 reports\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_cutadapt.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_data\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_counts\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_exp\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mcutadapt: deleting attribute self.cutadapt_length_obsexp\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.exec_modules                         \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRunning module: fastqc\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.modules.fastqc.fastqc                     \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Found 32 reports\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_fastqc.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.order_of_duplication_levels\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.base_module                               \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mfastqc: deleting attribute self.status_colours\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.software_versions                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReading software versions from config.software_versions\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_software_versions.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mRendering plots\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file multiqc_general_stats.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mExporting plot data to files\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_filtered_reads_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_trimmed_sequences_plot_3_Counts.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file cutadapt_trimmed_sequences_plot_3_Obs_Exp.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_counts_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_sequence_quality_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_quality_scores_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Percentages.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_sequence_gc_content_plot_Counts.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_per_base_n_content_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_length_distribution_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_sequence_duplication_levels_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_overrepresented_sequences_plot.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc_top_overrepresented_sequences_table.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.report                                    \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mWrote data file fastqc-status-check-heatmap.txt\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mMoving data file from '/tmp/tmpo_xhmpnl/multiqc_data' to '/opt/notebooks/03_multiqc_02_trim_galore/multiqc_data'\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Data        : 03_multiqc_02_trim_galore/multiqc_data\n",
      "\u001b[32m[2024-08-30 01:51:52]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mCompressing plot data\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:53]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  Report      : 03_multiqc_02_trim_galore/multiqc_report.html\n",
      "\u001b[32m[2024-08-30 01:51:53]\u001b[0m \u001b[34mmultiqc.core.write_results                        \u001b[0m \u001b[1;30m[DEBUG  ]\u001b[0m  \u001b[2mReport HTML written to /opt/notebooks/03_multiqc_02_trim_galore/multiqc_report.html\u001b[0m\n",
      "\u001b[32m[2024-08-30 01:51:53]\u001b[0m \u001b[34mmultiqc.multiqc                                   \u001b[0m \u001b[1;30m[INFO   ]\u001b[0m  MultiQC complete\n",
      "Fri Aug 30 01:51:53 UTC 2024\n"
     ]
    }
   ],
   "source": [
    "!date\n",
    "!multiqc 02_trim_galore -o 03_multiqc_02_trim_galore --verbose\n",
    "!date"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "91ba0348-2d27-4a2a-853e-855336a98b6e",
   "metadata": {},
   "source": [
    "---\n",
    "\n",
    "**NEXT:** BLAT\n",
    "\n",
    "---"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "27a80534-bd2e-4b09-b868-efd6687d652d",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Directory 240830_022731 created.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "# Get the current date and time in the format YYMMDD_HHMMSS\n",
    "current_time=$(date +\"%y%m%d_%H%M%S\")\n",
    "\n",
    "# Define the directory name\n",
    "dir_name=\"$current_time\"\n",
    "\n",
    "# Create the directory\n",
    "mkdir \"$dir_name\"\n",
    "\n",
    "echo \"Directory $dir_name created.\"\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cd1a5d96-c042-4997-b8f9-b3461df7b8b5",
   "metadata": {},
   "source": [
    "---\n",
    "\n",
    "To run each job in the `for` loop in the background, you can append an `&` to the command that initiates Trim Galore and use `wait` to ensure that the script waits for all background jobs to finish before exiting. This approach will allow each Trim Galore process to run concurrently.\n",
    "\n",
    "Heres how you can modify your script to run each job in the background:\n",
    "\n",
    "### **Modified Script**\n",
    "\n",
    "```bash\n",
    "#!/bin/bash\n",
    "\n",
    "# Define the base directory and output directory\n",
    "BASE_DIR=\"20240827_abundant_emu\"\n",
    "OUTPUT_DIR=\"02_trim_galore\"\n",
    "\n",
    "# Create the output directory if it doesn't exist\n",
    "mkdir -p $OUTPUT_DIR\n",
    "\n",
    "# Find all .fastq.gz files and run Trim Galore on them in the background\n",
    "find $BASE_DIR -type f -name \"*.fastq.gz\" | while read file; do\n",
    "    # Construct the output file name\n",
    "    base_name=$(basename \"$file\" .fastq.gz)\n",
    "    output_file=\"$OUTPUT_DIR/${base_name}_trimmed.fastq.gz\"\n",
    "    \n",
    "    # Run Trim Galore in the background\n",
    "    echo \"Running Trim Galore on $file\"\n",
    "    trim_galore \"$file\" --output_dir \"$OUTPUT_DIR\" --gzip &\n",
    "done\n",
    "\n",
    "# Wait for all background jobs to finish\n",
    "wait\n",
    "\n",
    "echo \"All Trim Galore processes are complete.\"\n",
    "```\n",
    "\n",
    "### **Explanation**:\n",
    "\n",
    "- **`&`**: Appending `&` at the end of the `trim_galore` command puts the process in the background, allowing the script to continue to the next iteration of the loop without waiting for the current job to finish.\n",
    "  \n",
    "- **`wait`**: This command is used to pause the script until all background jobs have completed. Without this, the script might exit before all `trim_galore` processes have finished.\n",
    "\n",
    "### **Considerations**:\n",
    "\n",
    "1. **Resource Management**: Running many jobs in parallel can put a significant load on your system. Make sure you have enough resources (CPU, memory) to handle the number of concurrent processes. If needed, you can limit the number of concurrent jobs by using tools like `xargs` with the `-P` option or job control mechanisms.\n",
    "\n",
    "2. **Error Handling**: You might want to add error handling to ensure that if a `trim_galore` process fails, it is reported. This can be done by capturing the exit status of each background job.\n",
    "\n",
    "By modifying your script this way, each instance of `trim_galore` will run concurrently, potentially speeding up the processing time if your system can handle the load.\n",
    "\n",
    "---"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5b5ff28c-3d66-4634-9d96-41195516d560",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "# SCRATCH"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dfbff42b-58a3-4aa5-ab4d-d9785624c40f",
   "metadata": {
    "tags": []
   },
   "source": [
    "---\n",
    "\n",
    "\n",
    "To run FastQC on all `.fastq.gz` files in the subfolders of a directory and output the results to a new folder, follow these steps:\n",
    "\n",
    "1. **Create the Output Directory**:\n",
    "   Ensure that the output directory `01_fastqc` exists or create it if it doesn't.\n",
    "\n",
    "2. **Run FastQC on All `.fastq.gz` Files**:\n",
    "   Use a loop to find all `.fastq.gz` files and run FastQC on each file, directing the output to the `01_fastqc` folder.\n",
    "\n",
    "Heres how you can do this with a shell script:\n",
    "\n",
    "### **Shell Script**\n",
    "\n",
    "1. **Open a Text Editor** and create a new shell script file, e.g., `run_fastqc.sh`.\n",
    "\n",
    "2. **Paste the Following Script** into your shell script file:\n",
    "\n",
    "   ```bash\n",
    "   #!/bin/bash\n",
    "\n",
    "   # Define the base directory and output directory\n",
    "   BASE_DIR=\"20240827_abundant_emu\"\n",
    "   OUTPUT_DIR=\"01_fastqc\"\n",
    "\n",
    "   # Create the output directory if it doesn't exist\n",
    "   mkdir -p $OUTPUT_DIR\n",
    "\n",
    "   # Find all .fastq.gz files and run FastQC on them\n",
    "   find $BASE_DIR -type f -name \"*.fastq.gz\" | while read file; do\n",
    "       echo \"Running FastQC on $file\"\n",
    "       fastqc \"$file\" --outdir=\"$OUTPUT_DIR\"\n",
    "   done\n",
    "   ```\n",
    "\n",
    "3. **Save the Script** and make it executable:\n",
    "\n",
    "   ```bash\n",
    "   chmod +x run_fastqc.sh\n",
    "   ```\n",
    "\n",
    "4. **Run the Script**:\n",
    "\n",
    "   ```bash\n",
    "   ./run_fastqc.sh\n",
    "   ```\n",
    "\n",
    "### **Explanation**:\n",
    "\n",
    "- `find $BASE_DIR -type f -name \"*.fastq.gz\"`: Searches for all files with the `.fastq.gz` extension in the specified base directory and its subfolders.\n",
    "- `while read file; do ... done`: Loops through each file found by `find`.\n",
    "- `fastqc \"$file\" --outdir=\"$OUTPUT_DIR\"`: Runs FastQC on each file and saves the output to the `01_fastqc` directory.\n",
    "\n",
    "### **Dependencies**:\n",
    "\n",
    "Ensure that FastQC is installed and available in your system's PATH. You can check this by running `fastqc --version`. If FastQC is not installed, you can download it from the [FastQC website](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and follow the installation instructions.\n",
    "\n",
    "By following these steps, you will efficiently run FastQC on all `.fastq.gz` files in the subfolders of your specified directory and store the results in a new output folder.\n",
    "\n",
    "---"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "83ee6eb8-354b-40b6-8f49-ced00cc82a7e",
   "metadata": {},
   "source": [
    "---\n",
    "\n",
    "To return the number of lines in all `.fastq.gz` files in the subfolders of a directory, you can use a combination of `find`, `zcat`, and `wc -l` commands in a shell script. Heres how you can accomplish this:\n",
    "\n",
    "### **Shell Script**\n",
    "\n",
    "1. **Open a Text Editor** and create a new shell script file, e.g., `count_lines.sh`.\n",
    "\n",
    "2. **Paste the Following Script** into your shell script file:\n",
    "\n",
    "   ```bash\n",
    "   #!/bin/bash\n",
    "\n",
    "   # Define the base directory\n",
    "   BASE_DIR=\"20240827_abundant_emu\"\n",
    "\n",
    "   # Find all .fastq.gz files and count the number of lines in each file\n",
    "   find $BASE_DIR -type f -name \"*.fastq.gz\" | while read file; do\n",
    "       # Count lines in the file\n",
    "       line_count=$(zcat \"$file\" | wc -l)\n",
    "       # Output the result\n",
    "       echo \"$file: $line_count lines\"\n",
    "   done\n",
    "   ```\n",
    "\n",
    "3. **Save the Script** and make it executable:\n",
    "\n",
    "   ```bash\n",
    "   chmod +x count_lines.sh\n",
    "   ```\n",
    "\n",
    "4. **Run the Script**:\n",
    "\n",
    "   ```bash\n",
    "   ./count_lines.sh\n",
    "   ```\n",
    "\n",
    "### **Explanation**:\n",
    "\n",
    "- `find $BASE_DIR -type f -name \"*.fastq.gz\"`: Searches for all files with the `.fastq.gz` extension in the specified base directory and its subfolders.\n",
    "- `while read file; do ... done`: Loops through each file found by `find`.\n",
    "- `zcat \"$file\" | wc -l`: `zcat` decompresses the `.fastq.gz` file to standard output, and `wc -l` counts the number of lines. This combination gives you the total number of lines in the file.\n",
    "- `echo \"$file: $line_count lines\"`: Outputs the file path and the number of lines.\n",
    "\n",
    "### **Running the Script**\n",
    "\n",
    "Make sure you have `zcat`, `find`, and `wc` installed and available in your systems PATH. These commands are typically available by default on most Unix-like systems.\n",
    "\n",
    "By running this script, you will get a count of the number of lines in each `.fastq.gz` file located in the subfolders of `20240827_abundant_emu`.\n",
    "\n",
    "\n",
    "---"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "205961d2-5932-47db-b7e3-cb1b1fd9b8a7",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
